// ngx-date-fns/esm2020/lib/add-seconds.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import addSeconds from 'date-fns/addSeconds';
import * as i0 from "@angular/core";
export class AddSecondsPipe {
    transform(date, amount) {
        return addSeconds(date, amount);
    }
}
AddSecondsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddSecondsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
AddSecondsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddSecondsPipe, name: "dfnsAddSeconds" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddSecondsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsAddSeconds' }]
        }] });
export class AddSecondsPipeModule {
}
AddSecondsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddSecondsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
AddSecondsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddSecondsPipeModule, declarations: [AddSecondsPipe], exports: [AddSecondsPipe] });
AddSecondsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddSecondsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddSecondsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [AddSecondsPipe],
                    exports: [AddSecondsPipe]
                }]
        }] });
