// ngx-date-fns/esm2020/lib/add-hours.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import addHours from 'date-fns/addHours';
import * as i0 from "@angular/core";
export class AddHoursPipe {
    transform(date, amount) {
        return addHours(date, amount);
    }
}
AddHoursPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddHoursPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
AddHoursPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddHoursPipe, name: "dfnsAddHours" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddHoursPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsAddHours' }]
        }] });
export class AddHoursPipeModule {
}
AddHoursPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddHoursPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
AddHoursPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddHoursPipeModule, declarations: [AddHoursPipe], exports: [AddHoursPipe] });
AddHoursPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddHoursPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddHoursPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [AddHoursPipe],
                    exports: [AddHoursPipe]
                }]
        }] });
