// ngx-date-fns/esm2020/lib/add-minutes.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import addMinutes from 'date-fns/addMinutes';
import * as i0 from "@angular/core";
export class AddMinutesPipe {
    transform(date, amount) {
        return addMinutes(date, amount);
    }
}
AddMinutesPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMinutesPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
AddMinutesPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMinutesPipe, name: "dfnsAddMinutes" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMinutesPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsAddMinutes' }]
        }] });
export class AddMinutesPipeModule {
}
AddMinutesPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMinutesPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
AddMinutesPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMinutesPipeModule, declarations: [AddMinutesPipe], exports: [AddMinutesPipe] });
AddMinutesPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMinutesPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMinutesPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [AddMinutesPipe],
                    exports: [AddMinutesPipe]
                }]
        }] });
