// ngx-date-fns/esm2020/lib/add-milliseconds.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import addMilliseconds from 'date-fns/addMilliseconds';
import * as i0 from "@angular/core";
export class AddMillisecondsPipe {
    transform(date, amount) {
        return addMilliseconds(date, amount);
    }
}
AddMillisecondsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMillisecondsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
AddMillisecondsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMillisecondsPipe, name: "dfnsAddMilliseconds" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMillisecondsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsAddMilliseconds' }]
        }] });
export class AddMillisecondsPipeModule {
}
AddMillisecondsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMillisecondsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
AddMillisecondsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMillisecondsPipeModule, declarations: [AddMillisecondsPipe], exports: [AddMillisecondsPipe] });
AddMillisecondsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMillisecondsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMillisecondsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [AddMillisecondsPipe],
                    exports: [AddMillisecondsPipe]
                }]
        }] });
