// ngx-date-fns/esm2020/lib/add-quarters.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import addQuarters from 'date-fns/addQuarters';
import * as i0 from "@angular/core";
export class AddQuartersPipe {
    transform(date, amount) {
        return addQuarters(date, amount);
    }
}
AddQuartersPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddQuartersPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
AddQuartersPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddQuartersPipe, name: "dfnsAddQuarters" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddQuartersPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsAddQuarters' }]
        }] });
export class AddQuartersPipeModule {
}
AddQuartersPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddQuartersPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
AddQuartersPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddQuartersPipeModule, declarations: [AddQuartersPipe], exports: [AddQuartersPipe] });
AddQuartersPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddQuartersPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddQuartersPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [AddQuartersPipe],
                    exports: [AddQuartersPipe]
                }]
        }] });
