// ngx-date-fns/esm2020/lib/add-iso-week-years.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import addISOWeekYears from 'date-fns/addISOWeekYears';
import * as i0 from "@angular/core";
export class AddISOWeekYearsPipe {
    transform(date, amount) {
        return addISOWeekYears(date, amount);
    }
}
AddISOWeekYearsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddISOWeekYearsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
AddISOWeekYearsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddISOWeekYearsPipe, name: "dfnsAddISOWeekYears" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddISOWeekYearsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsAddISOWeekYears' }]
        }] });
export class AddISOWeekYearsPipeModule {
}
AddISOWeekYearsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddISOWeekYearsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
AddISOWeekYearsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddISOWeekYearsPipeModule, declarations: [AddISOWeekYearsPipe], exports: [AddISOWeekYearsPipe] });
AddISOWeekYearsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddISOWeekYearsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddISOWeekYearsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [AddISOWeekYearsPipe],
                    exports: [AddISOWeekYearsPipe]
                }]
        }] });
