// ngx-date-fns/esm2020/lib/add-business-days.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import addBusinessDays from 'date-fns/addBusinessDays';
import * as i0 from "@angular/core";
export class AddBusinessDaysPipe {
    transform(date, amount) {
        return addBusinessDays(date, amount);
    }
}
AddBusinessDaysPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddBusinessDaysPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
AddBusinessDaysPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddBusinessDaysPipe, name: "dfnsAddBusinessDays" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddBusinessDaysPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsAddBusinessDays' }]
        }] });
export class AddBusinessDaysPipeModule {
}
AddBusinessDaysPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddBusinessDaysPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
AddBusinessDaysPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddBusinessDaysPipeModule, declarations: [AddBusinessDaysPipe], exports: [AddBusinessDaysPipe] });
AddBusinessDaysPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddBusinessDaysPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddBusinessDaysPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [AddBusinessDaysPipe],
                    exports: [AddBusinessDaysPipe]
                }]
        }] });
