// ngx-date-fns/esm2020/lib/add-years.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import addYears from 'date-fns/addYears';
import * as i0 from "@angular/core";
export class AddYearsPipe {
    transform(date, amount) {
        return addYears(date, amount);
    }
}
AddYearsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddYearsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
AddYearsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddYearsPipe, name: "dfnsAddYears" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddYearsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsAddYears' }]
        }] });
export class AddYearsPipeModule {
}
AddYearsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddYearsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
AddYearsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddYearsPipeModule, declarations: [AddYearsPipe], exports: [AddYearsPipe] });
AddYearsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddYearsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddYearsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [AddYearsPipe],
                    exports: [AddYearsPipe]
                }]
        }] });
