// ngx-date-fns/esm2020/lib/add-days.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import addDays from 'date-fns/addDays';
import * as i0 from "@angular/core";
export class AddDaysPipe {
    transform(date, amount) {
        return addDays(date, amount);
    }
}
AddDaysPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddDaysPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
AddDaysPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddDaysPipe, name: "dfnsAddDays" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddDaysPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsAddDays' }]
        }] });
export class AddDaysPipeModule {
}
AddDaysPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddDaysPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
AddDaysPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddDaysPipeModule, declarations: [AddDaysPipe], exports: [AddDaysPipe] });
AddDaysPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddDaysPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddDaysPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [AddDaysPipe],
                    exports: [AddDaysPipe]
                }]
        }] });
