// ngx-date-fns/esm2020/lib/add-months.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import addMonths from 'date-fns/addMonths';
import * as i0 from "@angular/core";
export class AddMonthsPipe {
    transform(date, amount) {
        return addMonths(date, amount);
    }
}
AddMonthsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMonthsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
AddMonthsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMonthsPipe, name: "dfnsAddMonths" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMonthsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsAddMonths' }]
        }] });
export class AddMonthsPipeModule {
}
AddMonthsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMonthsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
AddMonthsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMonthsPipeModule, declarations: [AddMonthsPipe], exports: [AddMonthsPipe] });
AddMonthsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMonthsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddMonthsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [AddMonthsPipe],
                    exports: [AddMonthsPipe]
                }]
        }] });
