// ngx-date-fns/esm2020/lib/add-weeks.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import addWeeks from 'date-fns/addWeeks';
import * as i0 from "@angular/core";
export class AddWeeksPipe {
    transform(date, amount) {
        return addWeeks(date, amount);
    }
}
AddWeeksPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddWeeksPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
AddWeeksPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddWeeksPipe, name: "dfnsAddWeeks" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddWeeksPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsAddWeeks' }]
        }] });
export class AddWeeksPipeModule {
}
AddWeeksPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddWeeksPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
AddWeeksPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddWeeksPipeModule, declarations: [AddWeeksPipe], exports: [AddWeeksPipe] });
AddWeeksPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddWeeksPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: AddWeeksPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [AddWeeksPipe],
                    exports: [AddWeeksPipe]
                }]
        }] });
