// ngx-date-fns/esm2020/lib/closest-to.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import closestTo from 'date-fns/closestTo';
import * as i0 from "@angular/core";
export class ClosestToPipe {
    transform(dateToCompare, datesArray) {
        return closestTo(dateToCompare, datesArray);
    }
}
ClosestToPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ClosestToPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
ClosestToPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ClosestToPipe, name: "dfnsClosestTo" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ClosestToPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsClosestTo' }]
        }] });
export class ClosestToPipeModule {
}
ClosestToPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ClosestToPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
ClosestToPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ClosestToPipeModule, declarations: [ClosestToPipe], exports: [ClosestToPipe] });
ClosestToPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ClosestToPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ClosestToPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [ClosestToPipe],
                    exports: [ClosestToPipe]
                }]
        }] });
