// ngx-date-fns/esm2020/lib/date-fns-configuration.service.mjs
import { Injectable } from '@angular/core';
import { Subject } from 'rxjs';
import * as i0 from "@angular/core";
export class DateFnsConfigurationService {
    constructor() {
        this.localeChanged = new Subject();
    }
    locale() {
        return this.locale$;
    }
    setLocale(locale) {
        this.locale$ = locale;
        this.localeChanged.next();
    }
}
DateFnsConfigurationService.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DateFnsConfigurationService, deps: [], target: i0.ɵɵFactoryTarget.Injectable });
DateFnsConfigurationService.ɵprov = i0.ɵɵngDeclareInjectable({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DateFnsConfigurationService, providedIn: 'root' });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DateFnsConfigurationService, decorators: [{
            type: Injectable,
            args: [{ providedIn: 'root' }]
        }] });
/**
 * Helper function used by all pipes to calculate locale
 */
export const calculateLocale = (options, config) => {
    const configLocale = config.locale();
    if (!options && configLocale) {
        return { locale: configLocale };
    }
    if (options && !options.locale && configLocale) {
        return {
            ...options,
            locale: configLocale
        };
    }
    return options;
};
