// ngx-date-fns/esm2020/lib/difference-in-months.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import differenceInMonths from 'date-fns/differenceInMonths';
import * as i0 from "@angular/core";
export class DifferenceInMonthsPipe {
    transform(dateLeft, dateRight) {
        return differenceInMonths(dateLeft, dateRight);
    }
}
DifferenceInMonthsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMonthsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
DifferenceInMonthsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMonthsPipe, name: "dfnsDifferenceInMonths" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMonthsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsDifferenceInMonths' }]
        }] });
export class DifferenceInMonthsPipeModule {
}
DifferenceInMonthsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMonthsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
DifferenceInMonthsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMonthsPipeModule, declarations: [DifferenceInMonthsPipe], exports: [DifferenceInMonthsPipe] });
DifferenceInMonthsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMonthsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMonthsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [DifferenceInMonthsPipe],
                    exports: [DifferenceInMonthsPipe]
                }]
        }] });
