// ngx-date-fns/esm2020/lib/difference-in-weeks.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import differenceInWeeks from 'date-fns/differenceInWeeks';
import * as i0 from "@angular/core";
export class DifferenceInWeeksPipe {
    transform(dateLeft, dateRight) {
        return differenceInWeeks(dateLeft, dateRight);
    }
}
DifferenceInWeeksPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInWeeksPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
DifferenceInWeeksPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInWeeksPipe, name: "dfnsDifferenceInWeeks" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInWeeksPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsDifferenceInWeeks' }]
        }] });
export class DifferenceInWeeksPipeModule {
}
DifferenceInWeeksPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInWeeksPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
DifferenceInWeeksPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInWeeksPipeModule, declarations: [DifferenceInWeeksPipe], exports: [DifferenceInWeeksPipe] });
DifferenceInWeeksPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInWeeksPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInWeeksPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [DifferenceInWeeksPipe],
                    exports: [DifferenceInWeeksPipe]
                }]
        }] });
