// ngx-date-fns/esm2020/lib/difference-in-calendar-days.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import differenceInCalendarDays from 'date-fns/differenceInCalendarDays';
import * as i0 from "@angular/core";
export class DifferenceInCalendarDaysPipe {
    transform(dateLeft, dateRight) {
        return differenceInCalendarDays(dateLeft, dateRight);
    }
}
DifferenceInCalendarDaysPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarDaysPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
DifferenceInCalendarDaysPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarDaysPipe, name: "dfnsDifferenceInCalendarDays" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarDaysPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsDifferenceInCalendarDays' }]
        }] });
export class DifferenceInCalendarDaysPipeModule {
}
DifferenceInCalendarDaysPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarDaysPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
DifferenceInCalendarDaysPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarDaysPipeModule, declarations: [DifferenceInCalendarDaysPipe], exports: [DifferenceInCalendarDaysPipe] });
DifferenceInCalendarDaysPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarDaysPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarDaysPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [DifferenceInCalendarDaysPipe],
                    exports: [DifferenceInCalendarDaysPipe]
                }]
        }] });
