// ngx-date-fns/esm2020/lib/difference-in-calendar-weeks.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import differenceInCalendarWeeks from 'date-fns/differenceInCalendarWeeks';
import * as i0 from "@angular/core";
export class DifferenceInCalendarWeeksPipe {
    transform(dateLeft, dateRight, options) {
        return differenceInCalendarWeeks(dateLeft, dateRight, options);
    }
}
DifferenceInCalendarWeeksPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarWeeksPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
DifferenceInCalendarWeeksPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarWeeksPipe, name: "dfnsDifferenceInCalendarWeeks" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarWeeksPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsDifferenceInCalendarWeeks' }]
        }] });
export class DifferenceInCalendarWeeksPipeModule {
}
DifferenceInCalendarWeeksPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarWeeksPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
DifferenceInCalendarWeeksPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarWeeksPipeModule, declarations: [DifferenceInCalendarWeeksPipe], exports: [DifferenceInCalendarWeeksPipe] });
DifferenceInCalendarWeeksPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarWeeksPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarWeeksPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [DifferenceInCalendarWeeksPipe],
                    exports: [DifferenceInCalendarWeeksPipe]
                }]
        }] });
