// ngx-date-fns/esm2020/lib/difference-in-calendar-iso-week-years.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import differenceInCalendarISOWeekYears from 'date-fns/differenceInCalendarISOWeekYears';
import * as i0 from "@angular/core";
export class DifferenceInCalendarISOWeekYearsPipe {
    transform(dateLeft, dateRight) {
        return differenceInCalendarISOWeekYears(dateLeft, dateRight);
    }
}
DifferenceInCalendarISOWeekYearsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarISOWeekYearsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
DifferenceInCalendarISOWeekYearsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarISOWeekYearsPipe, name: "dfnsDifferenceInCalendarISOWeekYears" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarISOWeekYearsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsDifferenceInCalendarISOWeekYears' }]
        }] });
export class DifferenceInCalendarISOWeekYearsPipeModule {
}
DifferenceInCalendarISOWeekYearsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarISOWeekYearsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
DifferenceInCalendarISOWeekYearsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarISOWeekYearsPipeModule, declarations: [DifferenceInCalendarISOWeekYearsPipe], exports: [DifferenceInCalendarISOWeekYearsPipe] });
DifferenceInCalendarISOWeekYearsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarISOWeekYearsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarISOWeekYearsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [DifferenceInCalendarISOWeekYearsPipe],
                    exports: [DifferenceInCalendarISOWeekYearsPipe]
                }]
        }] });
