// ngx-date-fns/esm2020/lib/difference-in-business-days.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import differenceInBusinessDays from 'date-fns/differenceInBusinessDays';
import * as i0 from "@angular/core";
export class DifferenceInBusinessDaysPipe {
    transform(dateLeft, dateRight) {
        return differenceInBusinessDays(dateLeft, dateRight);
    }
}
DifferenceInBusinessDaysPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInBusinessDaysPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
DifferenceInBusinessDaysPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInBusinessDaysPipe, name: "dfnsDifferenceInBusinessDays" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInBusinessDaysPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsDifferenceInBusinessDays' }]
        }] });
export class DifferenceInBusinessDaysPipeModule {
}
DifferenceInBusinessDaysPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInBusinessDaysPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
DifferenceInBusinessDaysPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInBusinessDaysPipeModule, declarations: [DifferenceInBusinessDaysPipe], exports: [DifferenceInBusinessDaysPipe] });
DifferenceInBusinessDaysPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInBusinessDaysPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInBusinessDaysPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [DifferenceInBusinessDaysPipe],
                    exports: [DifferenceInBusinessDaysPipe]
                }]
        }] });
