// ngx-date-fns/esm2020/lib/difference-in-days.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import differenceInDays from 'date-fns/differenceInDays';
import * as i0 from "@angular/core";
export class DifferenceInDaysPipe {
    transform(dateLeft, dateRight) {
        return differenceInDays(dateLeft, dateRight);
    }
}
DifferenceInDaysPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInDaysPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
DifferenceInDaysPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInDaysPipe, name: "dfnsDifferenceInDays" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInDaysPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsDifferenceInDays' }]
        }] });
export class DifferenceInDaysPipeModule {
}
DifferenceInDaysPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInDaysPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
DifferenceInDaysPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInDaysPipeModule, declarations: [DifferenceInDaysPipe], exports: [DifferenceInDaysPipe] });
DifferenceInDaysPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInDaysPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInDaysPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [DifferenceInDaysPipe],
                    exports: [DifferenceInDaysPipe]
                }]
        }] });
