// ngx-date-fns/esm2020/lib/difference-in-iso-week-years.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import differenceInISOWeekYears from 'date-fns/differenceInISOWeekYears';
import * as i0 from "@angular/core";
export class DifferenceInISOWeekYearsPipe {
    transform(dateLeft, dateRight) {
        return differenceInISOWeekYears(dateLeft, dateRight);
    }
}
DifferenceInISOWeekYearsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInISOWeekYearsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
DifferenceInISOWeekYearsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInISOWeekYearsPipe, name: "dfnsDifferenceInISOWeekYears" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInISOWeekYearsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsDifferenceInISOWeekYears' }]
        }] });
export class DifferenceInISOWeekYearsPipeModule {
}
DifferenceInISOWeekYearsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInISOWeekYearsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
DifferenceInISOWeekYearsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInISOWeekYearsPipeModule, declarations: [DifferenceInISOWeekYearsPipe], exports: [DifferenceInISOWeekYearsPipe] });
DifferenceInISOWeekYearsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInISOWeekYearsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInISOWeekYearsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [DifferenceInISOWeekYearsPipe],
                    exports: [DifferenceInISOWeekYearsPipe]
                }]
        }] });
