// ngx-date-fns/esm2020/lib/difference-in-calendar-quarters.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import differenceInCalendarQuarters from 'date-fns/differenceInCalendarQuarters';
import * as i0 from "@angular/core";
export class DifferenceInCalendarQuartersPipe {
    transform(dateLeft, dateRight) {
        return differenceInCalendarQuarters(dateLeft, dateRight);
    }
}
DifferenceInCalendarQuartersPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarQuartersPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
DifferenceInCalendarQuartersPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarQuartersPipe, name: "dfnsDifferenceInCalendarQuarters" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarQuartersPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsDifferenceInCalendarQuarters' }]
        }] });
export class DifferenceInCalendarQuartersPipeModule {
}
DifferenceInCalendarQuartersPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarQuartersPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
DifferenceInCalendarQuartersPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarQuartersPipeModule, declarations: [DifferenceInCalendarQuartersPipe], exports: [DifferenceInCalendarQuartersPipe] });
DifferenceInCalendarQuartersPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarQuartersPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarQuartersPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [DifferenceInCalendarQuartersPipe],
                    exports: [DifferenceInCalendarQuartersPipe]
                }]
        }] });
