// ngx-date-fns/esm2020/lib/difference-in-minutes.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import differenceInMinutes from 'date-fns/differenceInMinutes';
import * as i0 from "@angular/core";
export class DifferenceInMinutesPipe {
    transform(dateLeft, dateRight) {
        return differenceInMinutes(dateLeft, dateRight);
    }
}
DifferenceInMinutesPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMinutesPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
DifferenceInMinutesPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMinutesPipe, name: "dfnsDifferenceInMinutes" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMinutesPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsDifferenceInMinutes' }]
        }] });
export class DifferenceInMinutesPipeModule {
}
DifferenceInMinutesPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMinutesPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
DifferenceInMinutesPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMinutesPipeModule, declarations: [DifferenceInMinutesPipe], exports: [DifferenceInMinutesPipe] });
DifferenceInMinutesPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMinutesPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMinutesPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [DifferenceInMinutesPipe],
                    exports: [DifferenceInMinutesPipe]
                }]
        }] });
