// ngx-date-fns/esm2020/lib/difference-in-hours.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import differenceInHours from 'date-fns/differenceInHours';
import * as i0 from "@angular/core";
export class DifferenceInHoursPipe {
    transform(dateLeft, dateRight) {
        return differenceInHours(dateLeft, dateRight);
    }
}
DifferenceInHoursPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInHoursPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
DifferenceInHoursPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInHoursPipe, name: "dfnsDifferenceInHours" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInHoursPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsDifferenceInHours' }]
        }] });
export class DifferenceInHoursPipeModule {
}
DifferenceInHoursPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInHoursPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
DifferenceInHoursPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInHoursPipeModule, declarations: [DifferenceInHoursPipe], exports: [DifferenceInHoursPipe] });
DifferenceInHoursPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInHoursPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInHoursPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [DifferenceInHoursPipe],
                    exports: [DifferenceInHoursPipe]
                }]
        }] });
