// ngx-date-fns/esm2020/lib/difference-in-calendar-months.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import differenceInCalendarMonths from 'date-fns/differenceInCalendarMonths';
import * as i0 from "@angular/core";
export class DifferenceInCalendarMonthsPipe {
    transform(dateLeft, dateRight) {
        return differenceInCalendarMonths(dateLeft, dateRight);
    }
}
DifferenceInCalendarMonthsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarMonthsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
DifferenceInCalendarMonthsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarMonthsPipe, name: "dfnsDifferenceInCalendarMonths" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarMonthsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsDifferenceInCalendarMonths' }]
        }] });
export class DifferenceInCalendarMonthsPipeModule {
}
DifferenceInCalendarMonthsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarMonthsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
DifferenceInCalendarMonthsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarMonthsPipeModule, declarations: [DifferenceInCalendarMonthsPipe], exports: [DifferenceInCalendarMonthsPipe] });
DifferenceInCalendarMonthsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarMonthsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarMonthsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [DifferenceInCalendarMonthsPipe],
                    exports: [DifferenceInCalendarMonthsPipe]
                }]
        }] });
