// ngx-date-fns/esm2020/lib/difference-in-milliseconds.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import differenceInMilliseconds from 'date-fns/differenceInMilliseconds';
import * as i0 from "@angular/core";
export class DifferenceInMillisecondsPipe {
    transform(dateLeft, dateRight) {
        return differenceInMilliseconds(dateLeft, dateRight);
    }
}
DifferenceInMillisecondsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMillisecondsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
DifferenceInMillisecondsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMillisecondsPipe, name: "dfnsDifferenceInMilliseconds" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMillisecondsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsDifferenceInMilliseconds' }]
        }] });
export class DifferenceInMillisecondsPipeModule {
}
DifferenceInMillisecondsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMillisecondsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
DifferenceInMillisecondsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMillisecondsPipeModule, declarations: [DifferenceInMillisecondsPipe], exports: [DifferenceInMillisecondsPipe] });
DifferenceInMillisecondsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMillisecondsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInMillisecondsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [DifferenceInMillisecondsPipe],
                    exports: [DifferenceInMillisecondsPipe]
                }]
        }] });
