// ngx-date-fns/esm2020/lib/difference-in-years.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import differenceInYears from 'date-fns/differenceInYears';
import * as i0 from "@angular/core";
export class DifferenceInYearsPipe {
    transform(dateLeft, dateRight) {
        return differenceInYears(dateLeft, dateRight);
    }
}
DifferenceInYearsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInYearsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
DifferenceInYearsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInYearsPipe, name: "dfnsDifferenceInYears" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInYearsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsDifferenceInYears' }]
        }] });
export class DifferenceInYearsPipeModule {
}
DifferenceInYearsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInYearsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
DifferenceInYearsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInYearsPipeModule, declarations: [DifferenceInYearsPipe], exports: [DifferenceInYearsPipe] });
DifferenceInYearsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInYearsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInYearsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [DifferenceInYearsPipe],
                    exports: [DifferenceInYearsPipe]
                }]
        }] });
