// ngx-date-fns/esm2020/lib/difference-in-quarters.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import differenceInQuarters from 'date-fns/differenceInQuarters';
import * as i0 from "@angular/core";
export class DifferenceInQuartersPipe {
    transform(dateLeft, dateRight) {
        return differenceInQuarters(dateLeft, dateRight);
    }
}
DifferenceInQuartersPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInQuartersPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
DifferenceInQuartersPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInQuartersPipe, name: "dfnsDifferenceInQuarters" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInQuartersPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsDifferenceInQuarters' }]
        }] });
export class DifferenceInQuartersPipeModule {
}
DifferenceInQuartersPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInQuartersPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
DifferenceInQuartersPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInQuartersPipeModule, declarations: [DifferenceInQuartersPipe], exports: [DifferenceInQuartersPipe] });
DifferenceInQuartersPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInQuartersPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInQuartersPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [DifferenceInQuartersPipe],
                    exports: [DifferenceInQuartersPipe]
                }]
        }] });
