// ngx-date-fns/esm2020/lib/difference-in-calendar-years.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import differenceInCalendarYears from 'date-fns/differenceInCalendarYears';
import * as i0 from "@angular/core";
export class DifferenceInCalendarYearsPipe {
    transform(dateLeft, dateRight) {
        return differenceInCalendarYears(dateLeft, dateRight);
    }
}
DifferenceInCalendarYearsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarYearsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
DifferenceInCalendarYearsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarYearsPipe, name: "dfnsDifferenceInCalendarYears" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarYearsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsDifferenceInCalendarYears' }]
        }] });
export class DifferenceInCalendarYearsPipeModule {
}
DifferenceInCalendarYearsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarYearsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
DifferenceInCalendarYearsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarYearsPipeModule, declarations: [DifferenceInCalendarYearsPipe], exports: [DifferenceInCalendarYearsPipe] });
DifferenceInCalendarYearsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarYearsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: DifferenceInCalendarYearsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [DifferenceInCalendarYearsPipe],
                    exports: [DifferenceInCalendarYearsPipe]
                }]
        }] });
