// ngx-date-fns/esm2020/lib/end-of-week.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import endOfWeek from 'date-fns/endOfWeek';
import * as i0 from "@angular/core";
export class EndOfWeekPipe {
    transform(date, options) {
        return endOfWeek(date, options);
    }
}
EndOfWeekPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfWeekPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
EndOfWeekPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfWeekPipe, name: "dfnsEndOfWeek" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfWeekPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsEndOfWeek' }]
        }] });
export class EndOfWeekPipeModule {
}
EndOfWeekPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfWeekPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
EndOfWeekPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfWeekPipeModule, declarations: [EndOfWeekPipe], exports: [EndOfWeekPipe] });
EndOfWeekPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfWeekPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfWeekPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [EndOfWeekPipe],
                    exports: [EndOfWeekPipe]
                }]
        }] });
