// ngx-date-fns/esm2020/lib/end-of-iso-week.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import endOfISOWeek from 'date-fns/endOfISOWeek';
import * as i0 from "@angular/core";
export class EndOfISOWeekPipe {
    transform(date) {
        return endOfISOWeek(date);
    }
}
EndOfISOWeekPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfISOWeekPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
EndOfISOWeekPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfISOWeekPipe, name: "dfnsEndOfISOWeek" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfISOWeekPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsEndOfISOWeek' }]
        }] });
export class EndOfISOWeekPipeModule {
}
EndOfISOWeekPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfISOWeekPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
EndOfISOWeekPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfISOWeekPipeModule, declarations: [EndOfISOWeekPipe], exports: [EndOfISOWeekPipe] });
EndOfISOWeekPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfISOWeekPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfISOWeekPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [EndOfISOWeekPipe],
                    exports: [EndOfISOWeekPipe]
                }]
        }] });
