// ngx-date-fns/esm2020/lib/end-of-today.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import endOfToday from 'date-fns/endOfToday';
import * as i0 from "@angular/core";
export class EndOfTodayPipe {
    transform() {
        return endOfToday();
    }
}
EndOfTodayPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfTodayPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
EndOfTodayPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfTodayPipe, name: "dfnsEndOfToday" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfTodayPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsEndOfToday' }]
        }] });
export class EndOfTodayPipeModule {
}
EndOfTodayPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfTodayPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
EndOfTodayPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfTodayPipeModule, declarations: [EndOfTodayPipe], exports: [EndOfTodayPipe] });
EndOfTodayPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfTodayPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfTodayPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [EndOfTodayPipe],
                    exports: [EndOfTodayPipe]
                }]
        }] });
