// ngx-date-fns/esm2020/lib/end-of-hour.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import endOfHour from 'date-fns/endOfHour';
import * as i0 from "@angular/core";
export class EndOfHourPipe {
    transform(date) {
        return endOfHour(date);
    }
}
EndOfHourPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfHourPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
EndOfHourPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfHourPipe, name: "dfnsEndOfHour" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfHourPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsEndOfHour' }]
        }] });
export class EndOfHourPipeModule {
}
EndOfHourPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfHourPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
EndOfHourPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfHourPipeModule, declarations: [EndOfHourPipe], exports: [EndOfHourPipe] });
EndOfHourPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfHourPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfHourPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [EndOfHourPipe],
                    exports: [EndOfHourPipe]
                }]
        }] });
