// ngx-date-fns/esm2020/lib/end-of-quarter.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import endOfQuarter from 'date-fns/endOfQuarter';
import * as i0 from "@angular/core";
export class EndOfQuarterPipe {
    transform(date) {
        return endOfQuarter(date);
    }
}
EndOfQuarterPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfQuarterPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
EndOfQuarterPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfQuarterPipe, name: "dfnsEndOfQuarter" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfQuarterPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsEndOfQuarter' }]
        }] });
export class EndOfQuarterPipeModule {
}
EndOfQuarterPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfQuarterPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
EndOfQuarterPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfQuarterPipeModule, declarations: [EndOfQuarterPipe], exports: [EndOfQuarterPipe] });
EndOfQuarterPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfQuarterPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfQuarterPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [EndOfQuarterPipe],
                    exports: [EndOfQuarterPipe]
                }]
        }] });
