// ngx-date-fns/esm2020/lib/end-of-day.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import endOfDay from 'date-fns/endOfDay';
import * as i0 from "@angular/core";
export class EndOfDayPipe {
    transform(date) {
        return endOfDay(date);
    }
}
EndOfDayPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfDayPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
EndOfDayPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfDayPipe, name: "dfnsEndOfDay" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfDayPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsEndOfDay' }]
        }] });
export class EndOfDayPipeModule {
}
EndOfDayPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfDayPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
EndOfDayPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfDayPipeModule, declarations: [EndOfDayPipe], exports: [EndOfDayPipe] });
EndOfDayPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfDayPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfDayPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [EndOfDayPipe],
                    exports: [EndOfDayPipe]
                }]
        }] });
