// ngx-date-fns/esm2020/lib/end-of-second.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import endOfSecond from 'date-fns/endOfSecond';
import * as i0 from "@angular/core";
export class EndOfSecondPipe {
    transform(date) {
        return endOfSecond(date);
    }
}
EndOfSecondPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfSecondPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
EndOfSecondPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfSecondPipe, name: "dfnsEndOfSecond" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfSecondPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsEndOfSecond' }]
        }] });
export class EndOfSecondPipeModule {
}
EndOfSecondPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfSecondPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
EndOfSecondPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfSecondPipeModule, declarations: [EndOfSecondPipe], exports: [EndOfSecondPipe] });
EndOfSecondPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfSecondPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfSecondPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [EndOfSecondPipe],
                    exports: [EndOfSecondPipe]
                }]
        }] });
