// ngx-date-fns/esm2020/lib/end-of-iso-week-year.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import endOfISOWeekYear from 'date-fns/endOfISOWeekYear';
import * as i0 from "@angular/core";
export class EndOfISOWeekYearPipe {
    transform(date) {
        return endOfISOWeekYear(date);
    }
}
EndOfISOWeekYearPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfISOWeekYearPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
EndOfISOWeekYearPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfISOWeekYearPipe, name: "dfnsEndOfISOWeekYear" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfISOWeekYearPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsEndOfISOWeekYear' }]
        }] });
export class EndOfISOWeekYearPipeModule {
}
EndOfISOWeekYearPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfISOWeekYearPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
EndOfISOWeekYearPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfISOWeekYearPipeModule, declarations: [EndOfISOWeekYearPipe], exports: [EndOfISOWeekYearPipe] });
EndOfISOWeekYearPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfISOWeekYearPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfISOWeekYearPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [EndOfISOWeekYearPipe],
                    exports: [EndOfISOWeekYearPipe]
                }]
        }] });
