// ngx-date-fns/esm2020/lib/end-of-minute.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import endOfMinute from 'date-fns/endOfMinute';
import * as i0 from "@angular/core";
export class EndOfMinutePipe {
    transform(date) {
        return endOfMinute(date);
    }
}
EndOfMinutePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfMinutePipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
EndOfMinutePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfMinutePipe, name: "dfnsEndOfMinute" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfMinutePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsEndOfMinute' }]
        }] });
export class EndOfMinutePipeModule {
}
EndOfMinutePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfMinutePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
EndOfMinutePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfMinutePipeModule, declarations: [EndOfMinutePipe], exports: [EndOfMinutePipe] });
EndOfMinutePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfMinutePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfMinutePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [EndOfMinutePipe],
                    exports: [EndOfMinutePipe]
                }]
        }] });
