// ngx-date-fns/esm2020/lib/end-of-year.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import endOfYear from 'date-fns/endOfYear';
import * as i0 from "@angular/core";
export class EndOfYearPipe {
    transform(date) {
        return endOfYear(date);
    }
}
EndOfYearPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfYearPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
EndOfYearPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfYearPipe, name: "dfnsEndOfYear" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfYearPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsEndOfYear' }]
        }] });
export class EndOfYearPipeModule {
}
EndOfYearPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfYearPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
EndOfYearPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfYearPipeModule, declarations: [EndOfYearPipe], exports: [EndOfYearPipe] });
EndOfYearPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfYearPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfYearPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [EndOfYearPipe],
                    exports: [EndOfYearPipe]
                }]
        }] });
