// ngx-date-fns/esm2020/lib/end-of-yesterday.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import endOfYesterday from 'date-fns/endOfYesterday';
import * as i0 from "@angular/core";
export class EndOfYesterdayPipe {
    transform() {
        return endOfYesterday();
    }
}
EndOfYesterdayPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfYesterdayPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
EndOfYesterdayPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfYesterdayPipe, name: "dfnsEndOfYesterday" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfYesterdayPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsEndOfYesterday' }]
        }] });
export class EndOfYesterdayPipeModule {
}
EndOfYesterdayPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfYesterdayPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
EndOfYesterdayPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfYesterdayPipeModule, declarations: [EndOfYesterdayPipe], exports: [EndOfYesterdayPipe] });
EndOfYesterdayPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfYesterdayPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfYesterdayPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [EndOfYesterdayPipe],
                    exports: [EndOfYesterdayPipe]
                }]
        }] });
