// ngx-date-fns/esm2020/lib/end-of-month.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import endOfMonth from 'date-fns/endOfMonth';
import * as i0 from "@angular/core";
export class EndOfMonthPipe {
    transform(date) {
        return endOfMonth(date);
    }
}
EndOfMonthPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfMonthPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
EndOfMonthPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfMonthPipe, name: "dfnsEndOfMonth" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfMonthPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsEndOfMonth' }]
        }] });
export class EndOfMonthPipeModule {
}
EndOfMonthPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfMonthPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
EndOfMonthPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfMonthPipeModule, declarations: [EndOfMonthPipe], exports: [EndOfMonthPipe] });
EndOfMonthPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfMonthPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfMonthPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [EndOfMonthPipe],
                    exports: [EndOfMonthPipe]
                }]
        }] });
