// ngx-date-fns/esm2020/lib/end-of-tomorrow.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import endOfTomorrow from 'date-fns/endOfTomorrow';
import * as i0 from "@angular/core";
export class EndOfTomorrowPipe {
    transform() {
        return endOfTomorrow();
    }
}
EndOfTomorrowPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfTomorrowPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
EndOfTomorrowPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfTomorrowPipe, name: "dfnsEndOfTomorrow" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfTomorrowPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsEndOfTomorrow' }]
        }] });
export class EndOfTomorrowPipeModule {
}
EndOfTomorrowPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfTomorrowPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
EndOfTomorrowPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfTomorrowPipeModule, declarations: [EndOfTomorrowPipe], exports: [EndOfTomorrowPipe] });
EndOfTomorrowPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfTomorrowPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: EndOfTomorrowPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [EndOfTomorrowPipe],
                    exports: [EndOfTomorrowPipe]
                }]
        }] });
