// ngx-date-fns/esm2020/lib/format-relative.pipe.mjs
import { Pipe, NgModule } from '@angular/core';
import { calculateLocale } from './date-fns-configuration.service';
import formatRelative from 'date-fns/formatRelative';
import { isValidDate } from './utils';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class FormatRelativePipe {
    constructor(config, cd) {
        this.config = config;
        this.cd = cd;
        this.localeChanged$ = this.config.localeChanged.subscribe(_ => this.cd.markForCheck());
    }
    ngOnDestroy() {
        this.localeChanged$.unsubscribe();
    }
    transform(date, dateToCompare, options) {
        if (isValidDate(date) && isValidDate(dateToCompare)) {
            return formatRelative(date, dateToCompare, calculateLocale(options, this.config));
        }
        return '';
    }
}
FormatRelativePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativePipe, deps: [{ token: i1.DateFnsConfigurationService }, { token: i0.ChangeDetectorRef }], target: i0.ɵɵFactoryTarget.Pipe });
FormatRelativePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativePipe, name: "dfnsFormatRelative", pure: false });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsFormatRelative', pure: false }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }, { type: i0.ChangeDetectorRef }]; } });
export class FormatRelativePipeModule {
}
FormatRelativePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
FormatRelativePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativePipeModule, declarations: [FormatRelativePipe], exports: [FormatRelativePipe] });
FormatRelativePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [FormatRelativePipe],
                    exports: [FormatRelativePipe]
                }]
        }] });
