// ngx-date-fns/esm2020/lib/format-distance-to-now-strict.pipe.mjs
import { Pipe, NgModule } from '@angular/core';
import { calculateLocale } from './date-fns-configuration.service';
import formatDistanceToNowStrict from 'date-fns/formatDistanceToNowStrict';
import { isValidDate } from './utils';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class FormatDistanceToNowStrictPipe {
    constructor(config, cd) {
        this.config = config;
        this.cd = cd;
        this.localeChanged$ = this.config.localeChanged.subscribe(_ => this.cd.markForCheck());
    }
    ngOnDestroy() {
        this.localeChanged$.unsubscribe();
    }
    transform(date, options) {
        if (isValidDate(date)) {
            return formatDistanceToNowStrict(date, calculateLocale(options, this.config));
        }
        return '';
    }
}
FormatDistanceToNowStrictPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowStrictPipe, deps: [{ token: i1.DateFnsConfigurationService }, { token: i0.ChangeDetectorRef }], target: i0.ɵɵFactoryTarget.Pipe });
FormatDistanceToNowStrictPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowStrictPipe, name: "dfnsFormatDistanceToNowStrict", pure: false });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowStrictPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsFormatDistanceToNowStrict', pure: false }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }, { type: i0.ChangeDetectorRef }]; } });
export class FormatDistanceToNowStrictPipeModule {
}
FormatDistanceToNowStrictPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowStrictPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
FormatDistanceToNowStrictPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowStrictPipeModule, declarations: [FormatDistanceToNowStrictPipe], exports: [FormatDistanceToNowStrictPipe] });
FormatDistanceToNowStrictPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowStrictPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowStrictPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [FormatDistanceToNowStrictPipe],
                    exports: [FormatDistanceToNowStrictPipe]
                }]
        }] });
