// ngx-date-fns/esm2020/lib/format-duration.pipe.mjs
import { Pipe, NgModule } from '@angular/core';
import { calculateLocale } from './date-fns-configuration.service';
import formatDuration from 'date-fns/formatDuration';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class FormatDurationPipe {
    constructor(config, cd) {
        this.config = config;
        this.cd = cd;
        this.localeChanged$ = this.config.localeChanged.subscribe(_ => this.cd.markForCheck());
    }
    ngOnDestroy() {
        this.localeChanged$.unsubscribe();
    }
    transform(duration, options) {
        return formatDuration(duration, calculateLocale(options, this.config));
    }
}
FormatDurationPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDurationPipe, deps: [{ token: i1.DateFnsConfigurationService }, { token: i0.ChangeDetectorRef }], target: i0.ɵɵFactoryTarget.Pipe });
FormatDurationPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDurationPipe, name: "dfnsFormatDuration", pure: false });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDurationPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsFormatDuration', pure: false }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }, { type: i0.ChangeDetectorRef }]; } });
export class FormatDurationPipeModule {
}
FormatDurationPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDurationPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
FormatDurationPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDurationPipeModule, declarations: [FormatDurationPipe], exports: [FormatDurationPipe] });
FormatDurationPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDurationPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDurationPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [FormatDurationPipe],
                    exports: [FormatDurationPipe]
                }]
        }] });
