// ngx-date-fns/esm2020/lib/format-duration.pure.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import formatDuration from 'date-fns/formatDuration';
import { calculateLocale } from './date-fns-configuration.service';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class FormatDurationPurePipe {
    constructor(config) {
        this.config = config;
    }
    transform(duration, options) {
        return formatDuration(duration, calculateLocale(options, this.config));
    }
}
FormatDurationPurePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDurationPurePipe, deps: [{ token: i1.DateFnsConfigurationService }], target: i0.ɵɵFactoryTarget.Pipe });
FormatDurationPurePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDurationPurePipe, name: "dfnsFormatDurationPure" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDurationPurePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsFormatDurationPure' }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }]; } });
export class FormatDurationPurePipeModule {
}
FormatDurationPurePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDurationPurePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
FormatDurationPurePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDurationPurePipeModule, declarations: [FormatDurationPurePipe], exports: [FormatDurationPurePipe] });
FormatDurationPurePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDurationPurePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDurationPurePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [FormatDurationPurePipe],
                    exports: [FormatDurationPurePipe]
                }]
        }] });
