// ngx-date-fns/esm2020/lib/format-distance-to-now.pipe.mjs
import { Pipe, NgModule } from '@angular/core';
import { calculateLocale } from './date-fns-configuration.service';
import formatDistanceToNow from 'date-fns/formatDistanceToNow';
import { isValidDate } from './utils';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class FormatDistanceToNowPipe {
    constructor(config, cd) {
        this.config = config;
        this.cd = cd;
        this.localeChanged$ = this.config.localeChanged.subscribe(_ => this.cd.markForCheck());
    }
    ngOnDestroy() {
        this.localeChanged$.unsubscribe();
    }
    transform(date, options) {
        if (isValidDate(date)) {
            return formatDistanceToNow(date, calculateLocale(options, this.config));
        }
        return '';
    }
}
FormatDistanceToNowPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowPipe, deps: [{ token: i1.DateFnsConfigurationService }, { token: i0.ChangeDetectorRef }], target: i0.ɵɵFactoryTarget.Pipe });
FormatDistanceToNowPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowPipe, name: "dfnsFormatDistanceToNow", pure: false });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsFormatDistanceToNow', pure: false }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }, { type: i0.ChangeDetectorRef }]; } });
export class FormatDistanceToNowPipeModule {
}
FormatDistanceToNowPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
FormatDistanceToNowPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowPipeModule, declarations: [FormatDistanceToNowPipe], exports: [FormatDistanceToNowPipe] });
FormatDistanceToNowPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [FormatDistanceToNowPipe],
                    exports: [FormatDistanceToNowPipe]
                }]
        }] });
