// ngx-date-fns/esm2020/lib/format-distance-to-now-strict.pure.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import formatDistanceToNowStrict from 'date-fns/formatDistanceToNowStrict';
import { calculateLocale } from './date-fns-configuration.service';
import { isValidDate } from './utils';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class FormatDistanceToNowStrictPurePipe {
    constructor(config) {
        this.config = config;
    }
    transform(date, options) {
        if (isValidDate(date)) {
            return formatDistanceToNowStrict(date, calculateLocale(options, this.config));
        }
        return '';
    }
}
FormatDistanceToNowStrictPurePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowStrictPurePipe, deps: [{ token: i1.DateFnsConfigurationService }], target: i0.ɵɵFactoryTarget.Pipe });
FormatDistanceToNowStrictPurePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowStrictPurePipe, name: "dfnsFormatDistanceToNowStrictPure" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowStrictPurePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsFormatDistanceToNowStrictPure' }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }]; } });
export class FormatDistanceToNowStrictPurePipeModule {
}
FormatDistanceToNowStrictPurePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowStrictPurePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
FormatDistanceToNowStrictPurePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowStrictPurePipeModule, declarations: [FormatDistanceToNowStrictPurePipe], exports: [FormatDistanceToNowStrictPurePipe] });
FormatDistanceToNowStrictPurePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowStrictPurePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowStrictPurePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [FormatDistanceToNowStrictPurePipe],
                    exports: [FormatDistanceToNowStrictPurePipe]
                }]
        }] });
