// ngx-date-fns/esm2020/lib/format-distance-strict.pipe.mjs
import { Pipe, NgModule } from '@angular/core';
import { calculateLocale } from './date-fns-configuration.service';
import formatDistanceStrict from 'date-fns/formatDistanceStrict';
import { isValidDate } from './utils';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class FormatDistanceStrictPipe {
    constructor(config, cd) {
        this.config = config;
        this.cd = cd;
        this.localeChanged$ = this.config.localeChanged.subscribe(_ => this.cd.markForCheck());
    }
    ngOnDestroy() {
        this.localeChanged$.unsubscribe();
    }
    transform(date, dateToCompare, options) {
        if (isValidDate(date) && isValidDate(dateToCompare)) {
            return formatDistanceStrict(date, dateToCompare, calculateLocale(options, this.config));
        }
        return '';
    }
}
FormatDistanceStrictPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceStrictPipe, deps: [{ token: i1.DateFnsConfigurationService }, { token: i0.ChangeDetectorRef }], target: i0.ɵɵFactoryTarget.Pipe });
FormatDistanceStrictPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceStrictPipe, name: "dfnsFormatDistanceStrict", pure: false });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceStrictPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsFormatDistanceStrict', pure: false }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }, { type: i0.ChangeDetectorRef }]; } });
export class FormatDistanceStrictPipeModule {
}
FormatDistanceStrictPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceStrictPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
FormatDistanceStrictPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceStrictPipeModule, declarations: [FormatDistanceStrictPipe], exports: [FormatDistanceStrictPipe] });
FormatDistanceStrictPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceStrictPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceStrictPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [FormatDistanceStrictPipe],
                    exports: [FormatDistanceStrictPipe]
                }]
        }] });
