// ngx-date-fns/esm2020/lib/format.pure.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import format from 'date-fns/format';
import { calculateLocale } from './date-fns-configuration.service';
import { isValidDate } from './utils';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class FormatPurePipe {
    constructor(config) {
        this.config = config;
    }
    transform(date, dateFormat, options) {
        if (isValidDate(date)) {
            return format(date, dateFormat, calculateLocale(options, this.config));
        }
        return '';
    }
}
FormatPurePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatPurePipe, deps: [{ token: i1.DateFnsConfigurationService }], target: i0.ɵɵFactoryTarget.Pipe });
FormatPurePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatPurePipe, name: "dfnsFormatPure" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatPurePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsFormatPure' }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }]; } });
export class FormatPurePipeModule {
}
FormatPurePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatPurePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
FormatPurePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatPurePipeModule, declarations: [FormatPurePipe], exports: [FormatPurePipe] });
FormatPurePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatPurePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatPurePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [FormatPurePipe],
                    exports: [FormatPurePipe]
                }]
        }] });
