// ngx-date-fns/esm2020/lib/format-distance.pure.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import formatDistance from 'date-fns/formatDistance';
import { calculateLocale } from './date-fns-configuration.service';
import { isValidDate } from './utils';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class FormatDistancePurePipe {
    constructor(config) {
        this.config = config;
    }
    transform(date, dateToCompare, options) {
        if (isValidDate(date) && isValidDate(dateToCompare)) {
            return formatDistance(date, dateToCompare, calculateLocale(options, this.config));
        }
        return '';
    }
}
FormatDistancePurePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistancePurePipe, deps: [{ token: i1.DateFnsConfigurationService }], target: i0.ɵɵFactoryTarget.Pipe });
FormatDistancePurePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistancePurePipe, name: "dfnsFormatDistancePure" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistancePurePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsFormatDistancePure' }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }]; } });
export class FormatDistancePurePipeModule {
}
FormatDistancePurePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistancePurePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
FormatDistancePurePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistancePurePipeModule, declarations: [FormatDistancePurePipe], exports: [FormatDistancePurePipe] });
FormatDistancePurePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistancePurePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistancePurePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [FormatDistancePurePipe],
                    exports: [FormatDistancePurePipe]
                }]
        }] });
