// ngx-date-fns/esm2020/lib/format-distance-strict.pure.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import formatDistanceStrict from 'date-fns/formatDistanceStrict';
import { calculateLocale } from './date-fns-configuration.service';
import { isValidDate } from './utils';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class FormatDistanceStrictPurePipe {
    constructor(config) {
        this.config = config;
    }
    transform(date, dateToCompare, options) {
        if (isValidDate(date) && isValidDate(dateToCompare)) {
            return formatDistanceStrict(date, dateToCompare, calculateLocale(options, this.config));
        }
        return '';
    }
}
FormatDistanceStrictPurePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceStrictPurePipe, deps: [{ token: i1.DateFnsConfigurationService }], target: i0.ɵɵFactoryTarget.Pipe });
FormatDistanceStrictPurePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceStrictPurePipe, name: "dfnsFormatDistanceStrictPure" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceStrictPurePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsFormatDistanceStrictPure' }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }]; } });
export class FormatDistanceStrictPurePipeModule {
}
FormatDistanceStrictPurePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceStrictPurePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
FormatDistanceStrictPurePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceStrictPurePipeModule, declarations: [FormatDistanceStrictPurePipe], exports: [FormatDistanceStrictPurePipe] });
FormatDistanceStrictPurePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceStrictPurePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceStrictPurePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [FormatDistanceStrictPurePipe],
                    exports: [FormatDistanceStrictPurePipe]
                }]
        }] });
