// ngx-date-fns/esm2020/lib/format-distance-to-now.pure.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import formatDistanceToNow from 'date-fns/formatDistanceToNow';
import { calculateLocale } from './date-fns-configuration.service';
import { isValidDate } from './utils';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class FormatDistanceToNowPurePipe {
    constructor(config) {
        this.config = config;
    }
    transform(date, options) {
        if (isValidDate(date)) {
            return formatDistanceToNow(date, calculateLocale(options, this.config));
        }
        return '';
    }
}
FormatDistanceToNowPurePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowPurePipe, deps: [{ token: i1.DateFnsConfigurationService }], target: i0.ɵɵFactoryTarget.Pipe });
FormatDistanceToNowPurePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowPurePipe, name: "dfnsFormatDistanceToNowPure" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowPurePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsFormatDistanceToNowPure' }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }]; } });
export class FormatDistanceToNowPurePipeModule {
}
FormatDistanceToNowPurePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowPurePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
FormatDistanceToNowPurePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowPurePipeModule, declarations: [FormatDistanceToNowPurePipe], exports: [FormatDistanceToNowPurePipe] });
FormatDistanceToNowPurePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowPurePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatDistanceToNowPurePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [FormatDistanceToNowPurePipe],
                    exports: [FormatDistanceToNowPurePipe]
                }]
        }] });
