// ngx-date-fns/esm2020/lib/get-month.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getMonth from 'date-fns/getMonth';
import * as i0 from "@angular/core";
export class GetMonthPipe {
    transform(date) {
        return getMonth(date);
    }
}
GetMonthPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMonthPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetMonthPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMonthPipe, name: "dfnsGetMonth" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMonthPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetMonth' }]
        }] });
export class GetMonthPipeModule {
}
GetMonthPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMonthPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetMonthPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMonthPipeModule, declarations: [GetMonthPipe], exports: [GetMonthPipe] });
GetMonthPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMonthPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMonthPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetMonthPipe],
                    exports: [GetMonthPipe]
                }]
        }] });
