// ngx-date-fns/esm2020/lib/get-iso-week-year.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getISOWeekYear from 'date-fns/getISOWeekYear';
import * as i0 from "@angular/core";
export class GetISOWeekYearPipe {
    transform(date) {
        return getISOWeekYear(date);
    }
}
GetISOWeekYearPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeekYearPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetISOWeekYearPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeekYearPipe, name: "dfnsGetISOWeekYear" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeekYearPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetISOWeekYear' }]
        }] });
export class GetISOWeekYearPipeModule {
}
GetISOWeekYearPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeekYearPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetISOWeekYearPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeekYearPipeModule, declarations: [GetISOWeekYearPipe], exports: [GetISOWeekYearPipe] });
GetISOWeekYearPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeekYearPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeekYearPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetISOWeekYearPipe],
                    exports: [GetISOWeekYearPipe]
                }]
        }] });
