// ngx-date-fns/esm2020/lib/get-weeks-in-month.pure.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getWeeksInMonth from 'date-fns/getWeeksInMonth';
import { calculateLocale } from './date-fns-configuration.service';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class GetWeeksInMonthPurePipe {
    constructor(config) {
        this.config = config;
    }
    transform(date, options) {
        return getWeeksInMonth(date, calculateLocale(options, this.config));
    }
}
GetWeeksInMonthPurePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeeksInMonthPurePipe, deps: [{ token: i1.DateFnsConfigurationService }], target: i0.ɵɵFactoryTarget.Pipe });
GetWeeksInMonthPurePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeeksInMonthPurePipe, name: "dfnsGetWeeksInMonthPure" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeeksInMonthPurePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetWeeksInMonthPure' }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }]; } });
export class GetWeeksInMonthPurePipeModule {
}
GetWeeksInMonthPurePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeeksInMonthPurePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetWeeksInMonthPurePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeeksInMonthPurePipeModule, declarations: [GetWeeksInMonthPurePipe], exports: [GetWeeksInMonthPurePipe] });
GetWeeksInMonthPurePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeeksInMonthPurePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeeksInMonthPurePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetWeeksInMonthPurePipe],
                    exports: [GetWeeksInMonthPurePipe]
                }]
        }] });
