// ngx-date-fns/esm2020/lib/get-overlapping-days-in-intervals.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getOverlappingDaysInIntervals from 'date-fns/getOverlappingDaysInIntervals';
import * as i0 from "@angular/core";
export class GetOverlappingDaysInIntervalsPipe {
    transform(intervalLeft, intervalRight) {
        return getOverlappingDaysInIntervals(intervalLeft, intervalRight);
    }
}
GetOverlappingDaysInIntervalsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetOverlappingDaysInIntervalsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetOverlappingDaysInIntervalsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetOverlappingDaysInIntervalsPipe, name: "dfnsGetOverlappingDaysInIntervals" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetOverlappingDaysInIntervalsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetOverlappingDaysInIntervals' }]
        }] });
export class GetOverlappingDaysInIntervalsPipeModule {
}
GetOverlappingDaysInIntervalsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetOverlappingDaysInIntervalsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetOverlappingDaysInIntervalsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetOverlappingDaysInIntervalsPipeModule, declarations: [GetOverlappingDaysInIntervalsPipe], exports: [GetOverlappingDaysInIntervalsPipe] });
GetOverlappingDaysInIntervalsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetOverlappingDaysInIntervalsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetOverlappingDaysInIntervalsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetOverlappingDaysInIntervalsPipe],
                    exports: [GetOverlappingDaysInIntervalsPipe]
                }]
        }] });
