// ngx-date-fns/esm2020/lib/get-hours.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getHours from 'date-fns/getHours';
import * as i0 from "@angular/core";
export class GetHoursPipe {
    transform(date) {
        return getHours(date);
    }
}
GetHoursPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetHoursPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetHoursPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetHoursPipe, name: "dfnsGetHours" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetHoursPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetHours' }]
        }] });
export class GetHoursPipeModule {
}
GetHoursPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetHoursPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetHoursPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetHoursPipeModule, declarations: [GetHoursPipe], exports: [GetHoursPipe] });
GetHoursPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetHoursPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetHoursPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetHoursPipe],
                    exports: [GetHoursPipe]
                }]
        }] });
