// ngx-date-fns/esm2020/lib/get-iso-weeks-in-year.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getISOWeeksInYear from 'date-fns/getISOWeeksInYear';
import * as i0 from "@angular/core";
export class GetISOWeeksInYearPipe {
    transform(date) {
        return getISOWeeksInYear(date);
    }
}
GetISOWeeksInYearPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeeksInYearPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetISOWeeksInYearPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeeksInYearPipe, name: "dfnsGetISOWeeksInYear" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeeksInYearPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetISOWeeksInYear' }]
        }] });
export class GetISOWeeksInYearPipeModule {
}
GetISOWeeksInYearPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeeksInYearPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetISOWeeksInYearPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeeksInYearPipeModule, declarations: [GetISOWeeksInYearPipe], exports: [GetISOWeeksInYearPipe] });
GetISOWeeksInYearPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeeksInYearPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeeksInYearPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetISOWeeksInYearPipe],
                    exports: [GetISOWeeksInYearPipe]
                }]
        }] });
