// ngx-date-fns/esm2020/lib/get-minutes.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getMinutes from 'date-fns/getMinutes';
import * as i0 from "@angular/core";
export class GetMinutesPipe {
    transform(date) {
        return getMinutes(date);
    }
}
GetMinutesPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMinutesPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetMinutesPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMinutesPipe, name: "dfnsGetMinutes" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMinutesPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetMinutes' }]
        }] });
export class GetMinutesPipeModule {
}
GetMinutesPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMinutesPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetMinutesPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMinutesPipeModule, declarations: [GetMinutesPipe], exports: [GetMinutesPipe] });
GetMinutesPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMinutesPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMinutesPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetMinutesPipe],
                    exports: [GetMinutesPipe]
                }]
        }] });
