// ngx-date-fns/esm2020/lib/get-milliseconds.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getMilliseconds from 'date-fns/getMilliseconds';
import * as i0 from "@angular/core";
export class GetMillisecondsPipe {
    transform(date) {
        return getMilliseconds(date);
    }
}
GetMillisecondsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMillisecondsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetMillisecondsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMillisecondsPipe, name: "dfnsGetMilliseconds" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMillisecondsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetMilliseconds' }]
        }] });
export class GetMillisecondsPipeModule {
}
GetMillisecondsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMillisecondsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetMillisecondsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMillisecondsPipeModule, declarations: [GetMillisecondsPipe], exports: [GetMillisecondsPipe] });
GetMillisecondsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMillisecondsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetMillisecondsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetMillisecondsPipe],
                    exports: [GetMillisecondsPipe]
                }]
        }] });
