// ngx-date-fns/esm2020/lib/get-decade.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getDecade from 'date-fns/getDecade';
import * as i0 from "@angular/core";
export class GetDecadePipe {
    transform(date) {
        return getDecade(date);
    }
}
GetDecadePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDecadePipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetDecadePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDecadePipe, name: "dfnsGetDecade" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDecadePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetDecade' }]
        }] });
export class GetDecadePipeModule {
}
GetDecadePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDecadePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetDecadePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDecadePipeModule, declarations: [GetDecadePipe], exports: [GetDecadePipe] });
GetDecadePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDecadePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDecadePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetDecadePipe],
                    exports: [GetDecadePipe]
                }]
        }] });
