// ngx-date-fns/esm2020/lib/get-week.pure.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getWeek from 'date-fns/getWeek';
import { calculateLocale } from './date-fns-configuration.service';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class GetWeekPurePipe {
    constructor(config) {
        this.config = config;
    }
    transform(date, options) {
        return getWeek(date, calculateLocale(options, this.config));
    }
}
GetWeekPurePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekPurePipe, deps: [{ token: i1.DateFnsConfigurationService }], target: i0.ɵɵFactoryTarget.Pipe });
GetWeekPurePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekPurePipe, name: "dfnsGetWeekPure" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekPurePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetWeekPure' }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }]; } });
export class GetWeekPurePipeModule {
}
GetWeekPurePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekPurePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetWeekPurePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekPurePipeModule, declarations: [GetWeekPurePipe], exports: [GetWeekPurePipe] });
GetWeekPurePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekPurePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekPurePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetWeekPurePipe],
                    exports: [GetWeekPurePipe]
                }]
        }] });
