// ngx-date-fns/esm2020/lib/get-iso-day.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getISODay from 'date-fns/getISODay';
import * as i0 from "@angular/core";
export class GetISODayPipe {
    transform(date) {
        return getISODay(date);
    }
}
GetISODayPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISODayPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetISODayPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISODayPipe, name: "dfnsGetISODay" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISODayPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetISODay' }]
        }] });
export class GetISODayPipeModule {
}
GetISODayPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISODayPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetISODayPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISODayPipeModule, declarations: [GetISODayPipe], exports: [GetISODayPipe] });
GetISODayPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISODayPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISODayPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetISODayPipe],
                    exports: [GetISODayPipe]
                }]
        }] });
