// ngx-date-fns/esm2020/lib/get-day.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getDay from 'date-fns/getDay';
import * as i0 from "@angular/core";
export class GetDayPipe {
    transform(date) {
        return getDay(date);
    }
}
GetDayPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDayPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetDayPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDayPipe, name: "dfnsGetDay" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDayPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetDay' }]
        }] });
export class GetDayPipeModule {
}
GetDayPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDayPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetDayPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDayPipeModule, declarations: [GetDayPipe], exports: [GetDayPipe] });
GetDayPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDayPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDayPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetDayPipe],
                    exports: [GetDayPipe]
                }]
        }] });
