// ngx-date-fns/esm2020/lib/get-week-of-month.pipe.mjs
import { Pipe, NgModule } from '@angular/core';
import getWeekOfMonth from 'date-fns/getWeekOfMonth';
import { calculateLocale } from './date-fns-configuration.service';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class GetWeekOfMonthPipe {
    constructor(config, cd) {
        this.config = config;
        this.cd = cd;
        this.localeChanged$ = this.config.localeChanged.subscribe(_ => this.cd.markForCheck());
    }
    ngOnDestroy() {
        this.localeChanged$.unsubscribe();
    }
    transform(date, options) {
        return getWeekOfMonth(date, calculateLocale(options, this.config));
    }
}
GetWeekOfMonthPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekOfMonthPipe, deps: [{ token: i1.DateFnsConfigurationService }, { token: i0.ChangeDetectorRef }], target: i0.ɵɵFactoryTarget.Pipe });
GetWeekOfMonthPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekOfMonthPipe, name: "dfnsGetWeekOfMonth", pure: false });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekOfMonthPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetWeekOfMonth', pure: false }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }, { type: i0.ChangeDetectorRef }]; } });
export class GetWeekOfMonthPipeModule {
}
GetWeekOfMonthPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekOfMonthPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetWeekOfMonthPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekOfMonthPipeModule, declarations: [GetWeekOfMonthPipe], exports: [GetWeekOfMonthPipe] });
GetWeekOfMonthPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekOfMonthPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekOfMonthPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetWeekOfMonthPipe],
                    exports: [GetWeekOfMonthPipe]
                }]
        }] });
