// ngx-date-fns/esm2020/lib/get-iso-week.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getISOWeek from 'date-fns/getISOWeek';
import * as i0 from "@angular/core";
export class GetISOWeekPipe {
    transform(date) {
        return getISOWeek(date);
    }
}
GetISOWeekPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeekPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetISOWeekPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeekPipe, name: "dfnsGetISOWeek" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeekPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetISOWeek' }]
        }] });
export class GetISOWeekPipeModule {
}
GetISOWeekPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeekPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetISOWeekPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeekPipeModule, declarations: [GetISOWeekPipe], exports: [GetISOWeekPipe] });
GetISOWeekPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeekPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetISOWeekPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetISOWeekPipe],
                    exports: [GetISOWeekPipe]
                }]
        }] });
