// ngx-date-fns/esm2020/lib/get-week-year.pure.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getWeekYear from 'date-fns/getWeekYear';
import { calculateLocale } from './date-fns-configuration.service';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class GetWeekYearPurePipe {
    constructor(config) {
        this.config = config;
    }
    transform(date, options) {
        return getWeekYear(date, calculateLocale(options, this.config));
    }
}
GetWeekYearPurePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekYearPurePipe, deps: [{ token: i1.DateFnsConfigurationService }], target: i0.ɵɵFactoryTarget.Pipe });
GetWeekYearPurePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekYearPurePipe, name: "dfnsGetWeekYearPure" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekYearPurePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetWeekYearPure' }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }]; } });
export class GetWeekYearPurePipeModule {
}
GetWeekYearPurePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekYearPurePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetWeekYearPurePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekYearPurePipeModule, declarations: [GetWeekYearPurePipe], exports: [GetWeekYearPurePipe] });
GetWeekYearPurePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekYearPurePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeekYearPurePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetWeekYearPurePipe],
                    exports: [GetWeekYearPurePipe]
                }]
        }] });
