// ngx-date-fns/esm2020/lib/get-day-of-year.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getDayOfYear from 'date-fns/getDayOfYear';
import * as i0 from "@angular/core";
export class GetDayOfYearPipe {
    transform(date) {
        return getDayOfYear(date);
    }
}
GetDayOfYearPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDayOfYearPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetDayOfYearPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDayOfYearPipe, name: "dfnsGetDayOfYear" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDayOfYearPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetDayOfYear' }]
        }] });
export class GetDayOfYearPipeModule {
}
GetDayOfYearPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDayOfYearPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetDayOfYearPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDayOfYearPipeModule, declarations: [GetDayOfYearPipe], exports: [GetDayOfYearPipe] });
GetDayOfYearPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDayOfYearPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDayOfYearPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetDayOfYearPipe],
                    exports: [GetDayOfYearPipe]
                }]
        }] });
