// ngx-date-fns/esm2020/lib/get-weeks-in-month.pipe.mjs
import { Pipe, NgModule } from '@angular/core';
import getWeeksInMonth from 'date-fns/getWeeksInMonth';
import { calculateLocale } from './date-fns-configuration.service';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class GetWeeksInMonthPipe {
    constructor(config, cd) {
        this.config = config;
        this.cd = cd;
        this.localeChanged$ = this.config.localeChanged.subscribe(_ => this.cd.markForCheck());
    }
    ngOnDestroy() {
        this.localeChanged$.unsubscribe();
    }
    transform(date, options) {
        return getWeeksInMonth(date, calculateLocale(options, this.config));
    }
}
GetWeeksInMonthPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeeksInMonthPipe, deps: [{ token: i1.DateFnsConfigurationService }, { token: i0.ChangeDetectorRef }], target: i0.ɵɵFactoryTarget.Pipe });
GetWeeksInMonthPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeeksInMonthPipe, name: "dfnsGetWeeksInMonth", pure: false });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeeksInMonthPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetWeeksInMonth', pure: false }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }, { type: i0.ChangeDetectorRef }]; } });
export class GetWeeksInMonthPipeModule {
}
GetWeeksInMonthPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeeksInMonthPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetWeeksInMonthPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeeksInMonthPipeModule, declarations: [GetWeeksInMonthPipe], exports: [GetWeeksInMonthPipe] });
GetWeeksInMonthPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeeksInMonthPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetWeeksInMonthPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetWeeksInMonthPipe],
                    exports: [GetWeeksInMonthPipe]
                }]
        }] });
