// ngx-date-fns/esm2020/lib/get-quarter.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getQuarter from 'date-fns/getQuarter';
import * as i0 from "@angular/core";
export class GetQuarterPipe {
    transform(date) {
        return getQuarter(date);
    }
}
GetQuarterPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetQuarterPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetQuarterPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetQuarterPipe, name: "dfnsGetQuarter" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetQuarterPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetQuarter' }]
        }] });
export class GetQuarterPipeModule {
}
GetQuarterPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetQuarterPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetQuarterPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetQuarterPipeModule, declarations: [GetQuarterPipe], exports: [GetQuarterPipe] });
GetQuarterPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetQuarterPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetQuarterPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetQuarterPipe],
                    exports: [GetQuarterPipe]
                }]
        }] });
