// ngx-date-fns/esm2020/lib/get-days-in-year.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getDaysInYear from 'date-fns/getDaysInYear';
import * as i0 from "@angular/core";
export class GetDaysInYearPipe {
    transform(date) {
        return getDaysInYear(date);
    }
}
GetDaysInYearPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDaysInYearPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetDaysInYearPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDaysInYearPipe, name: "dfnsGetDaysInYear" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDaysInYearPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetDaysInYear' }]
        }] });
export class GetDaysInYearPipeModule {
}
GetDaysInYearPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDaysInYearPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetDaysInYearPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDaysInYearPipeModule, declarations: [GetDaysInYearPipe], exports: [GetDaysInYearPipe] });
GetDaysInYearPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDaysInYearPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDaysInYearPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetDaysInYearPipe],
                    exports: [GetDaysInYearPipe]
                }]
        }] });
