// ngx-date-fns/esm2020/lib/get-seconds.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getSeconds from 'date-fns/getSeconds';
import * as i0 from "@angular/core";
export class GetSecondsPipe {
    transform(date) {
        return getSeconds(date);
    }
}
GetSecondsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetSecondsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetSecondsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetSecondsPipe, name: "dfnsGetSeconds" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetSecondsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetSeconds' }]
        }] });
export class GetSecondsPipeModule {
}
GetSecondsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetSecondsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetSecondsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetSecondsPipeModule, declarations: [GetSecondsPipe], exports: [GetSecondsPipe] });
GetSecondsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetSecondsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetSecondsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetSecondsPipe],
                    exports: [GetSecondsPipe]
                }]
        }] });
