// ngx-date-fns/esm2020/lib/get-days-in-month.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getDaysInMonth from 'date-fns/getDaysInMonth';
import * as i0 from "@angular/core";
export class GetDaysInMonthPipe {
    transform(date) {
        return getDaysInMonth(date);
    }
}
GetDaysInMonthPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDaysInMonthPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetDaysInMonthPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDaysInMonthPipe, name: "dfnsGetDaysInMonth" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDaysInMonthPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetDaysInMonth' }]
        }] });
export class GetDaysInMonthPipeModule {
}
GetDaysInMonthPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDaysInMonthPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetDaysInMonthPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDaysInMonthPipeModule, declarations: [GetDaysInMonthPipe], exports: [GetDaysInMonthPipe] });
GetDaysInMonthPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDaysInMonthPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDaysInMonthPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetDaysInMonthPipe],
                    exports: [GetDaysInMonthPipe]
                }]
        }] });
