// ngx-date-fns/esm2020/lib/get-time.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getTime from 'date-fns/getTime';
import * as i0 from "@angular/core";
export class GetTimePipe {
    transform(date) {
        return getTime(date);
    }
}
GetTimePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetTimePipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetTimePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetTimePipe, name: "dfnsGetTime" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetTimePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetTime' }]
        }] });
export class GetTimePipeModule {
}
GetTimePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetTimePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetTimePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetTimePipeModule, declarations: [GetTimePipe], exports: [GetTimePipe] });
GetTimePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetTimePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetTimePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetTimePipe],
                    exports: [GetTimePipe]
                }]
        }] });
