// ngx-date-fns/esm2020/lib/get-date.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getDate from 'date-fns/getDate';
import * as i0 from "@angular/core";
export class GetDatePipe {
    transform(date) {
        return getDate(date);
    }
}
GetDatePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDatePipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetDatePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDatePipe, name: "dfnsGetDate" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDatePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetDate' }]
        }] });
export class GetDatePipeModule {
}
GetDatePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDatePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetDatePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDatePipeModule, declarations: [GetDatePipe], exports: [GetDatePipe] });
GetDatePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDatePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetDatePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetDatePipe],
                    exports: [GetDatePipe]
                }]
        }] });
