// ngx-date-fns/esm2020/lib/get-year.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getYear from 'date-fns/getYear';
import * as i0 from "@angular/core";
export class GetYearPipe {
    transform(date) {
        return getYear(date);
    }
}
GetYearPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetYearPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetYearPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetYearPipe, name: "dfnsGetYear" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetYearPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetYear' }]
        }] });
export class GetYearPipeModule {
}
GetYearPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetYearPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetYearPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetYearPipeModule, declarations: [GetYearPipe], exports: [GetYearPipe] });
GetYearPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetYearPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetYearPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetYearPipe],
                    exports: [GetYearPipe]
                }]
        }] });
