// ngx-date-fns/esm2020/lib/get-unix-time.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import getUnixTime from 'date-fns/getUnixTime';
import * as i0 from "@angular/core";
export class GetUnixTimePipe {
    transform(date) {
        return getUnixTime(date);
    }
}
GetUnixTimePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetUnixTimePipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
GetUnixTimePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetUnixTimePipe, name: "dfnsGetUnixTime" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetUnixTimePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsGetUnixTime' }]
        }] });
export class GetUnixTimePipeModule {
}
GetUnixTimePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetUnixTimePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
GetUnixTimePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetUnixTimePipeModule, declarations: [GetUnixTimePipe], exports: [GetUnixTimePipe] });
GetUnixTimePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetUnixTimePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: GetUnixTimePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [GetUnixTimePipe],
                    exports: [GetUnixTimePipe]
                }]
        }] });
