// ngx-date-fns/esm2020/lib/is-after.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isAfter from 'date-fns/isAfter';
import * as i0 from "@angular/core";
export class IsAfterPipe {
    transform(date, dateToCompare) {
        return isAfter(date, dateToCompare);
    }
}
IsAfterPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsAfterPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsAfterPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsAfterPipe, name: "dfnsIsAfter" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsAfterPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsIsAfter' }]
        }] });
export class IsAfterPipeModule {
}
IsAfterPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsAfterPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsAfterPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsAfterPipeModule, declarations: [IsAfterPipe], exports: [IsAfterPipe] });
IsAfterPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsAfterPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsAfterPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsAfterPipe],
                    exports: [IsAfterPipe]
                }]
        }] });
