// ngx-date-fns/esm2020/lib/is-before.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isBefore from 'date-fns/isBefore';
import * as i0 from "@angular/core";
export class IsBeforePipe {
    transform(date, dateToCompare) {
        return isBefore(date, dateToCompare);
    }
}
IsBeforePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsBeforePipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsBeforePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsBeforePipe, name: "dfnsIsBefore" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsBeforePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsIsBefore' }]
        }] });
export class IsBeforePipeModule {
}
IsBeforePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsBeforePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsBeforePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsBeforePipeModule, declarations: [IsBeforePipe], exports: [IsBeforePipe] });
IsBeforePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsBeforePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsBeforePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsBeforePipe],
                    exports: [IsBeforePipe]
                }]
        }] });
