// ngx-date-fns/esm2020/lib/is-date.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isDate from 'date-fns/isDate';
import * as i0 from "@angular/core";
export class IsDatePipe {
    transform(date) {
        return isDate(date);
    }
}
IsDatePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsDatePipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsDatePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsDatePipe, name: "dfnsIsDate" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsDatePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsIsDate' }]
        }] });
export class IsDatePipeModule {
}
IsDatePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsDatePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsDatePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsDatePipeModule, declarations: [IsDatePipe], exports: [IsDatePipe] });
IsDatePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsDatePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsDatePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsDatePipe],
                    exports: [IsDatePipe]
                }]
        }] });
