// ngx-date-fns/esm2020/lib/is-equal.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isEqual from 'date-fns/isEqual';
import * as i0 from "@angular/core";
export class IsEqualPipe {
    transform(dateLeft, dateRight) {
        return isEqual(dateLeft, dateRight);
    }
}
IsEqualPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsEqualPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsEqualPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsEqualPipe, name: "dfnsIsEqual" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsEqualPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsIsEqual' }]
        }] });
export class IsEqualPipeModule {
}
IsEqualPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsEqualPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsEqualPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsEqualPipeModule, declarations: [IsEqualPipe], exports: [IsEqualPipe] });
IsEqualPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsEqualPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsEqualPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsEqualPipe],
                    exports: [IsEqualPipe]
                }]
        }] });
