// ngx-date-fns/esm2020/lib/is-exists.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isExists from 'date-fns/isExists';
import * as i0 from "@angular/core";
export class IsExistsPipe {
    transform(year, month, day) {
        return isExists(year, month, day);
    }
}
IsExistsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsExistsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsExistsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsExistsPipe, name: "dfnsIsExists" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsExistsPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsExists'
                }]
        }] });
export class IsExistsPipeModule {
}
IsExistsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsExistsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsExistsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsExistsPipeModule, declarations: [IsExistsPipe], exports: [IsExistsPipe] });
IsExistsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsExistsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsExistsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsExistsPipe],
                    exports: [IsExistsPipe]
                }]
        }] });
