// ngx-date-fns/esm2020/lib/is-first-day-of-month.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isFirstDayOfMonth from 'date-fns/isFirstDayOfMonth';
import * as i0 from "@angular/core";
export class IsFirstDayOfMonthPipe {
    transform(date) {
        return isFirstDayOfMonth(date);
    }
}
IsFirstDayOfMonthPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFirstDayOfMonthPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsFirstDayOfMonthPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFirstDayOfMonthPipe, name: "dfnsIsFirstDayOfMonth" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFirstDayOfMonthPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsFirstDayOfMonth'
                }]
        }] });
export class IsFirstDayOfMonthPipeModule {
}
IsFirstDayOfMonthPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFirstDayOfMonthPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsFirstDayOfMonthPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFirstDayOfMonthPipeModule, declarations: [IsFirstDayOfMonthPipe], exports: [IsFirstDayOfMonthPipe] });
IsFirstDayOfMonthPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFirstDayOfMonthPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFirstDayOfMonthPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsFirstDayOfMonthPipe],
                    exports: [IsFirstDayOfMonthPipe]
                }]
        }] });
