// ngx-date-fns/esm2020/lib/is-future.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isFuture from 'date-fns/isFuture';
import * as i0 from "@angular/core";
export class IsFuturePipe {
    transform(date) {
        return isFuture(date);
    }
}
IsFuturePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFuturePipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsFuturePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFuturePipe, name: "dfnsIsFuture" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFuturePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsIsFuture' }]
        }] });
export class IsFuturePipeModule {
}
IsFuturePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFuturePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsFuturePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFuturePipeModule, declarations: [IsFuturePipe], exports: [IsFuturePipe] });
IsFuturePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFuturePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFuturePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsFuturePipe],
                    exports: [IsFuturePipe]
                }]
        }] });
