// ngx-date-fns/esm2020/lib/is-friday.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isFriday from 'date-fns/isFriday';
import * as i0 from "@angular/core";
export class IsFridayPipe {
    transform(date) {
        return isFriday(date);
    }
}
IsFridayPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFridayPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsFridayPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFridayPipe, name: "dfnsIsFriday" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFridayPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsFriday'
                }]
        }] });
export class IsFridayPipeModule {
}
IsFridayPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFridayPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsFridayPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFridayPipeModule, declarations: [IsFridayPipe], exports: [IsFridayPipe] });
IsFridayPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFridayPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsFridayPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsFridayPipe],
                    exports: [IsFridayPipe]
                }]
        }] });
