// ngx-date-fns/esm2020/lib/is-leap-year.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isLeapYear from 'date-fns/isLeapYear';
import * as i0 from "@angular/core";
export class IsLeapYearPipe {
    transform(date) {
        return isLeapYear(date);
    }
}
IsLeapYearPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsLeapYearPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsLeapYearPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsLeapYearPipe, name: "dfnsIsLeapYear" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsLeapYearPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsLeapYear'
                }]
        }] });
export class IsLeapYearPipeModule {
}
IsLeapYearPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsLeapYearPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsLeapYearPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsLeapYearPipeModule, declarations: [IsLeapYearPipe], exports: [IsLeapYearPipe] });
IsLeapYearPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsLeapYearPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsLeapYearPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsLeapYearPipe],
                    exports: [IsLeapYearPipe]
                }]
        }] });
