// ngx-date-fns/esm2020/lib/is-last-day-of-month.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isLastDayOfMonth from 'date-fns/isLastDayOfMonth';
import * as i0 from "@angular/core";
export class IsLastDayOfMonthPipe {
    transform(date) {
        return isLastDayOfMonth(date);
    }
}
IsLastDayOfMonthPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsLastDayOfMonthPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsLastDayOfMonthPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsLastDayOfMonthPipe, name: "dfnsIsLastDayOfMonth" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsLastDayOfMonthPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsLastDayOfMonth'
                }]
        }] });
export class IsLastDayOfMonthPipeModule {
}
IsLastDayOfMonthPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsLastDayOfMonthPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsLastDayOfMonthPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsLastDayOfMonthPipeModule, declarations: [IsLastDayOfMonthPipe], exports: [IsLastDayOfMonthPipe] });
IsLastDayOfMonthPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsLastDayOfMonthPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsLastDayOfMonthPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsLastDayOfMonthPipe],
                    exports: [IsLastDayOfMonthPipe]
                }]
        }] });
