// ngx-date-fns/esm2020/lib/is-match.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import { calculateLocale } from './date-fns-configuration.service';
import { isMatch } from 'date-fns';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class IsMatchPipe {
    constructor(config, cd) {
        this.config = config;
        this.cd = cd;
        this.localeChanged$ = this.config.localeChanged.subscribe(_ => this.cd.markForCheck());
    }
    ngOnDestroy() {
        this.localeChanged$.unsubscribe();
    }
    transform(dateString, formatString, options) {
        return isMatch(dateString, formatString, calculateLocale(options, this.config));
    }
}
IsMatchPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMatchPipe, deps: [{ token: i1.DateFnsConfigurationService }, { token: i0.ChangeDetectorRef }], target: i0.ɵɵFactoryTarget.Pipe });
IsMatchPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMatchPipe, name: "dfnsIsMatch", pure: false });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMatchPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsIsMatch', pure: false }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }, { type: i0.ChangeDetectorRef }]; } });
export class IsMatchPipeModule {
}
IsMatchPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMatchPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsMatchPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMatchPipeModule, declarations: [IsMatchPipe], exports: [IsMatchPipe] });
IsMatchPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMatchPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMatchPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsMatchPipe],
                    exports: [IsMatchPipe]
                }]
        }] });
