// ngx-date-fns/esm2020/lib/is-match.pure.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import { calculateLocale } from './date-fns-configuration.service';
import isMatch from 'date-fns/isMatch';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class IsMatchPurePipe {
    constructor(config) {
        this.config = config;
    }
    transform(dateString, formatString, options) {
        return isMatch(dateString, formatString, calculateLocale(options, this.config));
    }
}
IsMatchPurePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMatchPurePipe, deps: [{ token: i1.DateFnsConfigurationService }], target: i0.ɵɵFactoryTarget.Pipe });
IsMatchPurePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMatchPurePipe, name: "dfnsIsMatchPure" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMatchPurePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsIsMatchPure' }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }]; } });
export class IsMatchPurePipeModule {
}
IsMatchPurePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMatchPurePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsMatchPurePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMatchPurePipeModule, declarations: [IsMatchPurePipe], exports: [IsMatchPurePipe] });
IsMatchPurePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMatchPurePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMatchPurePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsMatchPurePipe],
                    exports: [IsMatchPurePipe]
                }]
        }] });
