// ngx-date-fns/esm2020/lib/is-monday.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isMonday from 'date-fns/isMonday';
import * as i0 from "@angular/core";
export class IsMondayPipe {
    transform(date) {
        return isMonday(date);
    }
}
IsMondayPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMondayPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsMondayPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMondayPipe, name: "dfnsIsMonday" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMondayPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsMonday'
                }]
        }] });
export class IsMondayPipeModule {
}
IsMondayPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMondayPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsMondayPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMondayPipeModule, declarations: [IsMondayPipe], exports: [IsMondayPipe] });
IsMondayPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMondayPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsMondayPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsMondayPipe],
                    exports: [IsMondayPipe]
                }]
        }] });
