// ngx-date-fns/esm2020/lib/is-past.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isPast from 'date-fns/isPast';
import * as i0 from "@angular/core";
export class IsPastPipe {
    transform(date) {
        return isPast(date);
    }
}
IsPastPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsPastPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsPastPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsPastPipe, name: "dfnsIsPast" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsPastPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsIsPast' }]
        }] });
export class IsPastPipeModule {
}
IsPastPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsPastPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsPastPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsPastPipeModule, declarations: [IsPastPipe], exports: [IsPastPipe] });
IsPastPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsPastPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsPastPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsPastPipe],
                    exports: [IsPastPipe]
                }]
        }] });
