// ngx-date-fns/esm2020/lib/is-saturday.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isSaturday from 'date-fns/isSaturday';
import * as i0 from "@angular/core";
export class IsSaturdayPipe {
    transform(date) {
        return isSaturday(date);
    }
}
IsSaturdayPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSaturdayPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsSaturdayPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSaturdayPipe, name: "dfnsIsSaturday" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSaturdayPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsSaturday'
                }]
        }] });
export class IsSaturdayPipeModule {
}
IsSaturdayPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSaturdayPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsSaturdayPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSaturdayPipeModule, declarations: [IsSaturdayPipe], exports: [IsSaturdayPipe] });
IsSaturdayPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSaturdayPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSaturdayPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsSaturdayPipe],
                    exports: [IsSaturdayPipe]
                }]
        }] });
