// ngx-date-fns/esm2020/lib/is-same-month.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isSameMonth from 'date-fns/isSameMonth';
import * as i0 from "@angular/core";
export class IsSameMonthPipe {
    transform(dateLeft, dateRight) {
        return isSameMonth(dateLeft, dateRight);
    }
}
IsSameMonthPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameMonthPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsSameMonthPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameMonthPipe, name: "dfnsIsSameMonth" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameMonthPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsIsSameMonth' }]
        }] });
export class IsSameMonthPipeModule {
}
IsSameMonthPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameMonthPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsSameMonthPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameMonthPipeModule, declarations: [IsSameMonthPipe], exports: [IsSameMonthPipe] });
IsSameMonthPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameMonthPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameMonthPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsSameMonthPipe],
                    exports: [IsSameMonthPipe]
                }]
        }] });
