// ngx-date-fns/esm2020/lib/is-sunday.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isSunday from 'date-fns/isSunday';
import * as i0 from "@angular/core";
export class IsSundayPipe {
    transform(date) {
        return isSunday(date);
    }
}
IsSundayPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSundayPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsSundayPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSundayPipe, name: "dfnsIsSunday" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSundayPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsSunday'
                }]
        }] });
export class IsSundayPipeModule {
}
IsSundayPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSundayPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsSundayPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSundayPipeModule, declarations: [IsSundayPipe], exports: [IsSundayPipe] });
IsSundayPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSundayPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSundayPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsSundayPipe],
                    exports: [IsSundayPipe]
                }]
        }] });
