// ngx-date-fns/esm2020/lib/is-same-quarter.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isSameQuarter from 'date-fns/isSameQuarter';
import * as i0 from "@angular/core";
export class IsSameQuarterPipe {
    transform(dateLeft, dateRight) {
        return isSameQuarter(dateLeft, dateRight);
    }
}
IsSameQuarterPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameQuarterPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsSameQuarterPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameQuarterPipe, name: "dfnsIsSameQuarter" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameQuarterPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsSameQuarter'
                }]
        }] });
export class IsSameQuarterPipeModule {
}
IsSameQuarterPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameQuarterPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsSameQuarterPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameQuarterPipeModule, declarations: [IsSameQuarterPipe], exports: [IsSameQuarterPipe] });
IsSameQuarterPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameQuarterPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameQuarterPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsSameQuarterPipe],
                    exports: [IsSameQuarterPipe]
                }]
        }] });
