// ngx-date-fns/esm2020/lib/is-same-year.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isSameYear from 'date-fns/isSameYear';
import * as i0 from "@angular/core";
export class IsSameYearPipe {
    transform(dateLeft, dateRight) {
        return isSameYear(dateLeft, dateRight);
    }
}
IsSameYearPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameYearPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsSameYearPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameYearPipe, name: "dfnsIsSameYear" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameYearPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsIsSameYear' }]
        }] });
export class IsSameYearPipeModule {
}
IsSameYearPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameYearPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsSameYearPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameYearPipeModule, declarations: [IsSameYearPipe], exports: [IsSameYearPipe] });
IsSameYearPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameYearPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameYearPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsSameYearPipe],
                    exports: [IsSameYearPipe]
                }]
        }] });
