// ngx-date-fns/esm2020/lib/is-same-second.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isSameSecond from 'date-fns/isSameSecond';
import * as i0 from "@angular/core";
export class IsSameSecondPipe {
    transform(dateLeft, dateRight) {
        return isSameSecond(dateLeft, dateRight);
    }
}
IsSameSecondPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameSecondPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsSameSecondPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameSecondPipe, name: "dfnsIsSameSecond" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameSecondPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsSameSecond'
                }]
        }] });
export class IsSameSecondPipeModule {
}
IsSameSecondPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameSecondPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsSameSecondPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameSecondPipeModule, declarations: [IsSameSecondPipe], exports: [IsSameSecondPipe] });
IsSameSecondPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameSecondPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameSecondPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsSameSecondPipe],
                    exports: [IsSameSecondPipe]
                }]
        }] });
