// ngx-date-fns/esm2020/lib/is-same-minute.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isSameDay from 'date-fns/isSameMinute';
import * as i0 from "@angular/core";
export class IsSameMinutePipe {
    transform(dateLeft, dateRight) {
        return isSameDay(dateLeft, dateRight);
    }
}
IsSameMinutePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameMinutePipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsSameMinutePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameMinutePipe, name: "dfnsIsSameMinute" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameMinutePipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsSameMinute'
                }]
        }] });
export class IsSameMinutePipeModule {
}
IsSameMinutePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameMinutePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsSameMinutePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameMinutePipeModule, declarations: [IsSameMinutePipe], exports: [IsSameMinutePipe] });
IsSameMinutePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameMinutePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameMinutePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsSameMinutePipe],
                    exports: [IsSameMinutePipe]
                }]
        }] });
