// ngx-date-fns/esm2020/lib/is-same-week.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import { calculateLocale } from './date-fns-configuration.service';
import isSameWeek from 'date-fns/isSameWeek';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class IsSameWeekPipe {
    constructor(config, cd) {
        this.config = config;
        this.cd = cd;
        this.localeChanged$ = this.config.localeChanged.subscribe(_ => this.cd.markForCheck());
    }
    ngOnDestroy() {
        this.localeChanged$.unsubscribe();
    }
    transform(dateLeft, dateRight, options) {
        return isSameWeek(dateLeft, dateRight, calculateLocale(options, this.config));
    }
}
IsSameWeekPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameWeekPipe, deps: [{ token: i1.DateFnsConfigurationService }, { token: i0.ChangeDetectorRef }], target: i0.ɵɵFactoryTarget.Pipe });
IsSameWeekPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameWeekPipe, name: "dfnsIsSameWeek", pure: false });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameWeekPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsIsSameWeek', pure: false }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }, { type: i0.ChangeDetectorRef }]; } });
export class IsSameWeekPipeModule {
}
IsSameWeekPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameWeekPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsSameWeekPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameWeekPipeModule, declarations: [IsSameWeekPipe], exports: [IsSameWeekPipe] });
IsSameWeekPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameWeekPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameWeekPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsSameWeekPipe],
                    exports: [IsSameWeekPipe]
                }]
        }] });
