// ngx-date-fns/esm2020/lib/is-same-day.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isSameDay from 'date-fns/isSameDay';
import * as i0 from "@angular/core";
export class IsSameDayPipe {
    transform(dateLeft, dateRight) {
        return isSameDay(dateLeft, dateRight);
    }
}
IsSameDayPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameDayPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsSameDayPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameDayPipe, name: "dfnsIsSameDay" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameDayPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsSameDay'
                }]
        }] });
export class IsSameDayPipeModule {
}
IsSameDayPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameDayPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsSameDayPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameDayPipeModule, declarations: [IsSameDayPipe], exports: [IsSameDayPipe] });
IsSameDayPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameDayPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameDayPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsSameDayPipe],
                    exports: [IsSameDayPipe]
                }]
        }] });
