// ngx-date-fns/esm2020/lib/is-same-hour.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isSameHour from 'date-fns/isSameHour';
import * as i0 from "@angular/core";
export class IsSameHourPipe {
    transform(dateLeft, dateRight) {
        return isSameHour(dateLeft, dateRight);
    }
}
IsSameHourPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameHourPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsSameHourPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameHourPipe, name: "dfnsIsSameHour" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameHourPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsSameHour'
                }]
        }] });
export class IsSameHourPipeModule {
}
IsSameHourPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameHourPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsSameHourPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameHourPipeModule, declarations: [IsSameHourPipe], exports: [IsSameHourPipe] });
IsSameHourPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameHourPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameHourPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsSameHourPipe],
                    exports: [IsSameHourPipe]
                }]
        }] });
