// ngx-date-fns/esm2020/lib/is-same-week.pure.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import { calculateLocale } from './date-fns-configuration.service';
import isSameWeek from 'date-fns/isSameWeek';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class IsSameWeekPurePipe {
    constructor(config) {
        this.config = config;
    }
    transform(dateLeft, dateRight, options) {
        return isSameWeek(dateLeft, dateRight, calculateLocale(options, this.config));
    }
}
IsSameWeekPurePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameWeekPurePipe, deps: [{ token: i1.DateFnsConfigurationService }], target: i0.ɵɵFactoryTarget.Pipe });
IsSameWeekPurePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameWeekPurePipe, name: "dfnsIsSameWeekPure" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameWeekPurePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsIsSameWeekPure' }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }]; } });
export class IsSameWeekPurePipeModule {
}
IsSameWeekPurePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameWeekPurePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsSameWeekPurePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameWeekPurePipeModule, declarations: [IsSameWeekPurePipe], exports: [IsSameWeekPurePipe] });
IsSameWeekPurePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameWeekPurePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameWeekPurePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsSameWeekPurePipe],
                    exports: [IsSameWeekPurePipe]
                }]
        }] });
