// ngx-date-fns/esm2020/lib/is-same-isoweek-year.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isSameISOWeekYear from 'date-fns/isSameISOWeekYear';
import * as i0 from "@angular/core";
export class IsSameISOWeekYearPipe {
    transform(dateLeft, dateRight) {
        return isSameISOWeekYear(dateLeft, dateRight);
    }
}
IsSameISOWeekYearPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameISOWeekYearPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsSameISOWeekYearPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameISOWeekYearPipe, name: "dfnsIsSameISOWeekYear" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameISOWeekYearPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsSameISOWeekYear'
                }]
        }] });
export class IsSameISOWeekYearPipeModule {
}
IsSameISOWeekYearPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameISOWeekYearPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsSameISOWeekYearPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameISOWeekYearPipeModule, declarations: [IsSameISOWeekYearPipe], exports: [IsSameISOWeekYearPipe] });
IsSameISOWeekYearPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameISOWeekYearPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameISOWeekYearPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsSameISOWeekYearPipe],
                    exports: [IsSameISOWeekYearPipe]
                }]
        }] });
