// ngx-date-fns/esm2020/lib/is-same-isoweek.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isSameISOWeek from 'date-fns/isSameISOWeek';
import * as i0 from "@angular/core";
export class IsSameISOWeekPipe {
    transform(dateLeft, dateRight) {
        return isSameISOWeek(dateLeft, dateRight);
    }
}
IsSameISOWeekPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameISOWeekPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsSameISOWeekPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameISOWeekPipe, name: "dfnsIsSameISOWeek" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameISOWeekPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsSameISOWeek'
                }]
        }] });
export class IsSameISOWeekPipeModule {
}
IsSameISOWeekPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameISOWeekPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsSameISOWeekPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameISOWeekPipeModule, declarations: [IsSameISOWeekPipe], exports: [IsSameISOWeekPipe] });
IsSameISOWeekPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameISOWeekPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsSameISOWeekPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsSameISOWeekPipe],
                    exports: [IsSameISOWeekPipe]
                }]
        }] });
