// ngx-date-fns/esm2020/lib/is-tuesday.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isTuesday from 'date-fns/isTuesday';
import * as i0 from "@angular/core";
export class IsTuesdayPipe {
    transform(date) {
        return isTuesday(date);
    }
}
IsTuesdayPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTuesdayPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsTuesdayPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTuesdayPipe, name: "dfnsIsTuesday" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTuesdayPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsTuesday'
                }]
        }] });
export class IsTuesdayPipeModule {
}
IsTuesdayPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTuesdayPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsTuesdayPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTuesdayPipeModule, declarations: [IsTuesdayPipe], exports: [IsTuesdayPipe] });
IsTuesdayPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTuesdayPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTuesdayPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsTuesdayPipe],
                    exports: [IsTuesdayPipe]
                }]
        }] });
