// ngx-date-fns/esm2020/lib/is-this-quarter.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isThisQuarter from 'date-fns/isThisQuarter';
import * as i0 from "@angular/core";
export class IsThisQuarterPipe {
    transform(date) {
        return isThisQuarter(date);
    }
}
IsThisQuarterPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisQuarterPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsThisQuarterPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisQuarterPipe, name: "dfnsIsThisQuarter" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisQuarterPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsThisQuarter'
                }]
        }] });
export class IsThisQuarterPipeModule {
}
IsThisQuarterPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisQuarterPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsThisQuarterPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisQuarterPipeModule, declarations: [IsThisQuarterPipe], exports: [IsThisQuarterPipe] });
IsThisQuarterPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisQuarterPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisQuarterPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsThisQuarterPipe],
                    exports: [IsThisQuarterPipe]
                }]
        }] });
