// ngx-date-fns/esm2020/lib/is-thursday.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isThursday from 'date-fns/isThursday';
import * as i0 from "@angular/core";
export class IsThursdayPipe {
    transform(date) {
        return isThursday(date);
    }
}
IsThursdayPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThursdayPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsThursdayPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThursdayPipe, name: "dfnsIsThursday" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThursdayPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsThursday'
                }]
        }] });
export class IsThursdayPipeModule {
}
IsThursdayPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThursdayPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsThursdayPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThursdayPipeModule, declarations: [IsThursdayPipe], exports: [IsThursdayPipe] });
IsThursdayPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThursdayPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThursdayPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsThursdayPipe],
                    exports: [IsThursdayPipe]
                }]
        }] });
