// ngx-date-fns/esm2020/lib/is-this-minute.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isThisMinute from 'date-fns/isThisMinute';
import * as i0 from "@angular/core";
export class IsThisMinutePipe {
    transform(date) {
        return isThisMinute(date);
    }
}
IsThisMinutePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisMinutePipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsThisMinutePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisMinutePipe, name: "dfnsIsThisMinute" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisMinutePipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsThisMinute'
                }]
        }] });
export class IsThisMinutePipeModule {
}
IsThisMinutePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisMinutePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsThisMinutePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisMinutePipeModule, declarations: [IsThisMinutePipe], exports: [IsThisMinutePipe] });
IsThisMinutePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisMinutePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisMinutePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsThisMinutePipe],
                    exports: [IsThisMinutePipe]
                }]
        }] });
