// ngx-date-fns/esm2020/lib/is-this-hour.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isThisHour from 'date-fns/isThisHour';
import * as i0 from "@angular/core";
export class IsThisHourPipe {
    transform(date) {
        return isThisHour(date);
    }
}
IsThisHourPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisHourPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsThisHourPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisHourPipe, name: "dfnsIsThisHour" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisHourPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsThisHour'
                }]
        }] });
export class IsThisHourPipeModule {
}
IsThisHourPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisHourPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsThisHourPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisHourPipeModule, declarations: [IsThisHourPipe], exports: [IsThisHourPipe] });
IsThisHourPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisHourPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisHourPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsThisHourPipe],
                    exports: [IsThisHourPipe]
                }]
        }] });
