// ngx-date-fns/esm2020/lib/is-this-week.pure.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import { calculateLocale } from './date-fns-configuration.service';
import isThisWeek from 'date-fns/isThisWeek';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class IsThisWeekPurePipe {
    constructor(config) {
        this.config = config;
    }
    transform(date, options) {
        return isThisWeek(date, calculateLocale(options, this.config));
    }
}
IsThisWeekPurePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisWeekPurePipe, deps: [{ token: i1.DateFnsConfigurationService }], target: i0.ɵɵFactoryTarget.Pipe });
IsThisWeekPurePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisWeekPurePipe, name: "dfnsIsThisWeekPure" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisWeekPurePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsIsThisWeekPure' }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }]; } });
export class IsThisWeekPurePipeModule {
}
IsThisWeekPurePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisWeekPurePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsThisWeekPurePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisWeekPurePipeModule, declarations: [IsThisWeekPurePipe], exports: [IsThisWeekPurePipe] });
IsThisWeekPurePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisWeekPurePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisWeekPurePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsThisWeekPurePipe],
                    exports: [IsThisWeekPurePipe]
                }]
        }] });
