// ngx-date-fns/esm2020/lib/is-today.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isToday from 'date-fns/isToday';
import * as i0 from "@angular/core";
export class IsTodayPipe {
    transform(date) {
        return isToday(date);
    }
}
IsTodayPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTodayPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsTodayPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTodayPipe, name: "dfnsIsToday" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTodayPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsIsToday' }]
        }] });
export class IsTodayPipeModule {
}
IsTodayPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTodayPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsTodayPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTodayPipeModule, declarations: [IsTodayPipe], exports: [IsTodayPipe] });
IsTodayPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTodayPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTodayPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsTodayPipe],
                    exports: [IsTodayPipe]
                }]
        }] });
