// ngx-date-fns/esm2020/lib/is-this-isoweek.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isThisISOWeek from 'date-fns/isThisISOWeek';
import * as i0 from "@angular/core";
export class IsThisISOWeekPipe {
    transform(date) {
        return isThisISOWeek(date);
    }
}
IsThisISOWeekPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisISOWeekPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsThisISOWeekPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisISOWeekPipe, name: "dfnsIsThisISOWeek" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisISOWeekPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsThisISOWeek'
                }]
        }] });
export class IsThisISOWeekPipeModule {
}
IsThisISOWeekPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisISOWeekPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsThisISOWeekPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisISOWeekPipeModule, declarations: [IsThisISOWeekPipe], exports: [IsThisISOWeekPipe] });
IsThisISOWeekPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisISOWeekPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisISOWeekPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsThisISOWeekPipe],
                    exports: [IsThisISOWeekPipe]
                }]
        }] });
