// ngx-date-fns/esm2020/lib/is-tomorrow.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isTomorrow from 'date-fns/isTomorrow';
import * as i0 from "@angular/core";
export class IsTomorrowPipe {
    transform(date) {
        return isTomorrow(date);
    }
}
IsTomorrowPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTomorrowPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsTomorrowPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTomorrowPipe, name: "dfnsIsTomorrow" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTomorrowPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsTomorrow'
                }]
        }] });
export class IsTomorrowPipeModule {
}
IsTomorrowPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTomorrowPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsTomorrowPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTomorrowPipeModule, declarations: [IsTomorrowPipe], exports: [IsTomorrowPipe] });
IsTomorrowPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTomorrowPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsTomorrowPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsTomorrowPipe],
                    exports: [IsTomorrowPipe]
                }]
        }] });
