// ngx-date-fns/esm2020/lib/is-this-second.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isThisSecond from 'date-fns/isThisSecond';
import * as i0 from "@angular/core";
export class IsThisSecondPipe {
    transform(date) {
        return isThisSecond(date);
    }
}
IsThisSecondPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisSecondPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsThisSecondPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisSecondPipe, name: "dfnsIsThisSecond" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisSecondPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsThisSecond'
                }]
        }] });
export class IsThisSecondPipeModule {
}
IsThisSecondPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisSecondPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsThisSecondPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisSecondPipeModule, declarations: [IsThisSecondPipe], exports: [IsThisSecondPipe] });
IsThisSecondPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisSecondPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisSecondPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsThisSecondPipe],
                    exports: [IsThisSecondPipe]
                }]
        }] });
