// ngx-date-fns/esm2020/lib/is-this-week.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import { calculateLocale } from './date-fns-configuration.service';
import { isThisWeek } from 'date-fns';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class IsThisWeekPipe {
    constructor(config, cd) {
        this.config = config;
        this.cd = cd;
        this.localeChanged$ = this.config.localeChanged.subscribe(_ => this.cd.markForCheck());
    }
    ngOnDestroy() {
        this.localeChanged$.unsubscribe();
    }
    transform(date, options) {
        return isThisWeek(date, calculateLocale(options, this.config));
    }
}
IsThisWeekPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisWeekPipe, deps: [{ token: i1.DateFnsConfigurationService }, { token: i0.ChangeDetectorRef }], target: i0.ɵɵFactoryTarget.Pipe });
IsThisWeekPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisWeekPipe, name: "dfnsIsThisWeek", pure: false });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisWeekPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsIsThisWeek', pure: false }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }, { type: i0.ChangeDetectorRef }]; } });
export class IsThisWeekPipeModule {
}
IsThisWeekPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisWeekPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsThisWeekPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisWeekPipeModule, declarations: [IsThisWeekPipe], exports: [IsThisWeekPipe] });
IsThisWeekPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisWeekPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisWeekPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsThisWeekPipe],
                    exports: [IsThisWeekPipe]
                }]
        }] });
