// ngx-date-fns/esm2020/lib/is-this-year.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isThisYear from 'date-fns/isThisYear';
import * as i0 from "@angular/core";
export class IsThisYearPipe {
    transform(date) {
        return isThisYear(date);
    }
}
IsThisYearPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisYearPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsThisYearPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisYearPipe, name: "dfnsIsThisYear" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisYearPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsThisYear'
                }]
        }] });
export class IsThisYearPipeModule {
}
IsThisYearPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisYearPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsThisYearPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisYearPipeModule, declarations: [IsThisYearPipe], exports: [IsThisYearPipe] });
IsThisYearPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisYearPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisYearPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsThisYearPipe],
                    exports: [IsThisYearPipe]
                }]
        }] });
