// ngx-date-fns/esm2020/lib/is-this-month.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isThisMonth from 'date-fns/isThisMonth';
import * as i0 from "@angular/core";
export class IsThisMonthPipe {
    transform(date) {
        return isThisMonth(date);
    }
}
IsThisMonthPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisMonthPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsThisMonthPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisMonthPipe, name: "dfnsIsThisMonth" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisMonthPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsThisMonth'
                }]
        }] });
export class IsThisMonthPipeModule {
}
IsThisMonthPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisMonthPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsThisMonthPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisMonthPipeModule, declarations: [IsThisMonthPipe], exports: [IsThisMonthPipe] });
IsThisMonthPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisMonthPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsThisMonthPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsThisMonthPipe],
                    exports: [IsThisMonthPipe]
                }]
        }] });
