// ngx-date-fns/esm2020/lib/is-valid.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isValid from 'date-fns/isValid';
import * as i0 from "@angular/core";
export class IsValidPipe {
    transform(date) {
        return isValid(date);
    }
}
IsValidPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsValidPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsValidPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsValidPipe, name: "dfnsIsValid" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsValidPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsIsValid' }]
        }] });
export class IsValidPipeModule {
}
IsValidPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsValidPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsValidPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsValidPipeModule, declarations: [IsValidPipe], exports: [IsValidPipe] });
IsValidPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsValidPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsValidPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsValidPipe],
                    exports: [IsValidPipe]
                }]
        }] });
