// ngx-date-fns/esm2020/lib/is-wednesday.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isWednesday from 'date-fns/isWednesday';
import * as i0 from "@angular/core";
export class IsWednesdayPipe {
    transform(date) {
        return isWednesday(date);
    }
}
IsWednesdayPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWednesdayPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsWednesdayPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWednesdayPipe, name: "dfnsIsWednesday" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWednesdayPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsWednesday'
                }]
        }] });
export class IsWednesdayPipeModule {
}
IsWednesdayPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWednesdayPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsWednesdayPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWednesdayPipeModule, declarations: [IsWednesdayPipe], exports: [IsWednesdayPipe] });
IsWednesdayPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWednesdayPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWednesdayPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsWednesdayPipe],
                    exports: [IsWednesdayPipe]
                }]
        }] });
