// ngx-date-fns/esm2020/lib/is-within-interval.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isWithinInterval from 'date-fns/isWithinInterval';
import * as i0 from "@angular/core";
export class IsWithinIntervalPipe {
    transform(date, interval) {
        return isWithinInterval(date, interval);
    }
}
IsWithinIntervalPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWithinIntervalPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsWithinIntervalPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWithinIntervalPipe, name: "dfnsIsWithinInterval" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWithinIntervalPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsWithinInterval'
                }]
        }] });
export class IsWithinIntervalPipeModule {
}
IsWithinIntervalPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWithinIntervalPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsWithinIntervalPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWithinIntervalPipeModule, declarations: [IsWithinIntervalPipe], exports: [IsWithinIntervalPipe] });
IsWithinIntervalPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWithinIntervalPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWithinIntervalPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsWithinIntervalPipe],
                    exports: [IsWithinIntervalPipe]
                }]
        }] });
