// ngx-date-fns/esm2020/lib/is-weekend.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isWeekend from 'date-fns/isWeekend';
import * as i0 from "@angular/core";
export class IsWeekendPipe {
    transform(date) {
        return isWeekend(date);
    }
}
IsWeekendPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWeekendPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsWeekendPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWeekendPipe, name: "dfnsIsWeekend" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWeekendPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsIsWeekend' }]
        }] });
export class IsWeekendPipeModule {
}
IsWeekendPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWeekendPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsWeekendPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWeekendPipeModule, declarations: [IsWeekendPipe], exports: [IsWeekendPipe] });
IsWeekendPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWeekendPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsWeekendPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsWeekendPipe],
                    exports: [IsWeekendPipe]
                }]
        }] });
