// ngx-date-fns/esm2020/lib/is-yesterday.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import isYesterday from 'date-fns/isYesterday';
import * as i0 from "@angular/core";
export class IsYesterdayPipe {
    transform(date) {
        return isYesterday(date);
    }
}
IsYesterdayPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsYesterdayPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
IsYesterdayPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsYesterdayPipe, name: "dfnsIsYesterday" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsYesterdayPipe, decorators: [{
            type: Pipe,
            args: [{
                    name: 'dfnsIsYesterday'
                }]
        }] });
export class IsYesterdayPipeModule {
}
IsYesterdayPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsYesterdayPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
IsYesterdayPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsYesterdayPipeModule, declarations: [IsYesterdayPipe], exports: [IsYesterdayPipe] });
IsYesterdayPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsYesterdayPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: IsYesterdayPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [IsYesterdayPipe],
                    exports: [IsYesterdayPipe]
                }]
        }] });
