// ngx-date-fns/esm2020/lib/last-day-of-quarter.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import lastDayOfQuarter from 'date-fns/lastDayOfQuarter';
import * as i0 from "@angular/core";
export class LastDayOfQuarterPipe {
    transform(date) {
        return lastDayOfQuarter(date);
    }
}
LastDayOfQuarterPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfQuarterPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
LastDayOfQuarterPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfQuarterPipe, name: "dfnsLastDayOfQuarter" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfQuarterPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsLastDayOfQuarter' }]
        }] });
export class LastDayOfQuarterPipeModule {
}
LastDayOfQuarterPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfQuarterPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
LastDayOfQuarterPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfQuarterPipeModule, declarations: [LastDayOfQuarterPipe], exports: [LastDayOfQuarterPipe] });
LastDayOfQuarterPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfQuarterPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfQuarterPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [LastDayOfQuarterPipe],
                    exports: [LastDayOfQuarterPipe]
                }]
        }] });
