// ngx-date-fns/esm2020/lib/last-day-of-week.pure.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import lastDayOfWeek from 'date-fns/lastDayOfWeek';
import { calculateLocale } from './date-fns-configuration.service';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class LastDayOfWeekPurePipe {
    constructor(config) {
        this.config = config;
    }
    transform(date, options) {
        return lastDayOfWeek(date, calculateLocale(options, this.config));
    }
}
LastDayOfWeekPurePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfWeekPurePipe, deps: [{ token: i1.DateFnsConfigurationService }], target: i0.ɵɵFactoryTarget.Pipe });
LastDayOfWeekPurePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfWeekPurePipe, name: "dfnsLastDayOfWeekPure" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfWeekPurePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsLastDayOfWeekPure' }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }]; } });
export class LastDayOfWeekPurePipeModule {
}
LastDayOfWeekPurePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfWeekPurePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
LastDayOfWeekPurePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfWeekPurePipeModule, declarations: [LastDayOfWeekPurePipe], exports: [LastDayOfWeekPurePipe] });
LastDayOfWeekPurePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfWeekPurePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfWeekPurePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [LastDayOfWeekPurePipe],
                    exports: [LastDayOfWeekPurePipe]
                }]
        }] });
