// ngx-date-fns/esm2020/lib/last-day-of-decade.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import lastDayOfDecade from 'date-fns/lastDayOfDecade';
import * as i0 from "@angular/core";
export class LastDayOfDecadePipe {
    transform(date) {
        return lastDayOfDecade(date);
    }
}
LastDayOfDecadePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfDecadePipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
LastDayOfDecadePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfDecadePipe, name: "dfnsLastDayOfDecade" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfDecadePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsLastDayOfDecade' }]
        }] });
export class LastDayOfDecadePipeModule {
}
LastDayOfDecadePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfDecadePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
LastDayOfDecadePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfDecadePipeModule, declarations: [LastDayOfDecadePipe], exports: [LastDayOfDecadePipe] });
LastDayOfDecadePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfDecadePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfDecadePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [LastDayOfDecadePipe],
                    exports: [LastDayOfDecadePipe]
                }]
        }] });
