// ngx-date-fns/esm2020/lib/last-day-of-iso-week-year.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import lastDayOfISOWeekYear from 'date-fns/lastDayOfISOWeekYear';
import * as i0 from "@angular/core";
export class LastDayOfISOWeekYearPipe {
    transform(date) {
        return lastDayOfISOWeekYear(date);
    }
}
LastDayOfISOWeekYearPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfISOWeekYearPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
LastDayOfISOWeekYearPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfISOWeekYearPipe, name: "dfnsLastDayOfISOWeekYear" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfISOWeekYearPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsLastDayOfISOWeekYear' }]
        }] });
export class LastDayOfISOWeekYearPipeModule {
}
LastDayOfISOWeekYearPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfISOWeekYearPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
LastDayOfISOWeekYearPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfISOWeekYearPipeModule, declarations: [LastDayOfISOWeekYearPipe], exports: [LastDayOfISOWeekYearPipe] });
LastDayOfISOWeekYearPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfISOWeekYearPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfISOWeekYearPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [LastDayOfISOWeekYearPipe],
                    exports: [LastDayOfISOWeekYearPipe]
                }]
        }] });
