// ngx-date-fns/esm2020/lib/last-day-of-iso-week.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import lastDayOfISOWeek from 'date-fns/lastDayOfISOWeek';
import * as i0 from "@angular/core";
export class LastDayOfISOWeekPipe {
    transform(date) {
        return lastDayOfISOWeek(date);
    }
}
LastDayOfISOWeekPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfISOWeekPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
LastDayOfISOWeekPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfISOWeekPipe, name: "dfnsLastDayOfISOWeek" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfISOWeekPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsLastDayOfISOWeek' }]
        }] });
export class LastDayOfISOWeekPipeModule {
}
LastDayOfISOWeekPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfISOWeekPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
LastDayOfISOWeekPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfISOWeekPipeModule, declarations: [LastDayOfISOWeekPipe], exports: [LastDayOfISOWeekPipe] });
LastDayOfISOWeekPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfISOWeekPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfISOWeekPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [LastDayOfISOWeekPipe],
                    exports: [LastDayOfISOWeekPipe]
                }]
        }] });
