// ngx-date-fns/esm2020/lib/last-day-of-year.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import lastDayOfYear from 'date-fns/lastDayOfYear';
import * as i0 from "@angular/core";
export class LastDayOfYearPipe {
    transform(date) {
        return lastDayOfYear(date);
    }
}
LastDayOfYearPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfYearPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
LastDayOfYearPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfYearPipe, name: "dfnsLastDayOfYear" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfYearPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsLastDayOfYear' }]
        }] });
export class LastDayOfYearPipeModule {
}
LastDayOfYearPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfYearPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
LastDayOfYearPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfYearPipeModule, declarations: [LastDayOfYearPipe], exports: [LastDayOfYearPipe] });
LastDayOfYearPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfYearPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfYearPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [LastDayOfYearPipe],
                    exports: [LastDayOfYearPipe]
                }]
        }] });
