// ngx-date-fns/esm2020/lib/last-day-of-week.pipe.mjs
import { Pipe, NgModule } from '@angular/core';
import lastDayOfWeek from 'date-fns/lastDayOfWeek';
import { calculateLocale } from './date-fns-configuration.service';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class LastDayOfWeekPipe {
    constructor(config, cd) {
        this.config = config;
        this.cd = cd;
        this.localeChanged$ = this.config.localeChanged.subscribe(_ => this.cd.markForCheck());
    }
    ngOnDestroy() {
        this.localeChanged$.unsubscribe();
    }
    transform(date, options) {
        return lastDayOfWeek(date, calculateLocale(options, this.config));
    }
}
LastDayOfWeekPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfWeekPipe, deps: [{ token: i1.DateFnsConfigurationService }, { token: i0.ChangeDetectorRef }], target: i0.ɵɵFactoryTarget.Pipe });
LastDayOfWeekPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfWeekPipe, name: "dfnsLastDayOfWeek", pure: false });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfWeekPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsLastDayOfWeek', pure: false }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }, { type: i0.ChangeDetectorRef }]; } });
export class LastDayOfWeekPipeModule {
}
LastDayOfWeekPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfWeekPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
LastDayOfWeekPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfWeekPipeModule, declarations: [LastDayOfWeekPipe], exports: [LastDayOfWeekPipe] });
LastDayOfWeekPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfWeekPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfWeekPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [LastDayOfWeekPipe],
                    exports: [LastDayOfWeekPipe]
                }]
        }] });
