// ngx-date-fns/esm2020/lib/last-day-of-month.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import lastDayOfMonth from 'date-fns/lastDayOfMonth';
import * as i0 from "@angular/core";
export class LastDayOfMonthPipe {
    transform(date) {
        return lastDayOfMonth(date);
    }
}
LastDayOfMonthPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfMonthPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
LastDayOfMonthPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfMonthPipe, name: "dfnsLastDayOfMonth" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfMonthPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsLastDayOfMonth' }]
        }] });
export class LastDayOfMonthPipeModule {
}
LastDayOfMonthPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfMonthPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
LastDayOfMonthPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfMonthPipeModule, declarations: [LastDayOfMonthPipe], exports: [LastDayOfMonthPipe] });
LastDayOfMonthPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfMonthPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: LastDayOfMonthPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [LastDayOfMonthPipe],
                    exports: [LastDayOfMonthPipe]
                }]
        }] });
