// ngx-date-fns/esm2020/lib/max.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import max from 'date-fns/max';
import * as i0 from "@angular/core";
export class MaxPipe {
    transform(dates) {
        return max(dates);
    }
}
MaxPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: MaxPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
MaxPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: MaxPipe, name: "dfnsMax" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: MaxPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsMax' }]
        }] });
export class MaxPipeModule {
}
MaxPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: MaxPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
MaxPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: MaxPipeModule, declarations: [MaxPipe], exports: [MaxPipe] });
MaxPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: MaxPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: MaxPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [MaxPipe],
                    exports: [MaxPipe]
                }]
        }] });
