// ngx-date-fns/esm2020/lib/min.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import min from 'date-fns/min';
import * as i0 from "@angular/core";
export class MinPipe {
    transform(dates) {
        return min(dates);
    }
}
MinPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: MinPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
MinPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: MinPipe, name: "dfnsMin" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: MinPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsMin' }]
        }] });
export class MinPipeModule {
}
MinPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: MinPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
MinPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: MinPipeModule, declarations: [MinPipe], exports: [MinPipe] });
MinPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: MinPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: MinPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [MinPipe],
                    exports: [MinPipe]
                }]
        }] });
