// ngx-date-fns/esm2020/lib/parse.pure.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import parse from 'date-fns/parse';
import { calculateLocale } from './date-fns-configuration.service';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class ParsePurePipe {
    constructor(config) {
        this.config = config;
    }
    transform(dateString, formatString, backupDate, options) {
        return parse(dateString, formatString, backupDate, calculateLocale(options, this.config));
    }
}
ParsePurePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ParsePurePipe, deps: [{ token: i1.DateFnsConfigurationService }], target: i0.ɵɵFactoryTarget.Pipe });
ParsePurePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ParsePurePipe, name: "dfnsParsePure" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ParsePurePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsParsePure' }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }]; } });
export class ParsePurePipeModule {
}
ParsePurePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ParsePurePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
ParsePurePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ParsePurePipeModule, declarations: [ParsePurePipe], exports: [ParsePurePipe] });
ParsePurePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ParsePurePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ParsePurePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [ParsePurePipe],
                    exports: [ParsePurePipe]
                }]
        }] });
