// ngx-date-fns/esm2020/lib/parse-iso.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import parseISO from 'date-fns/parseISO';
import * as i0 from "@angular/core";
export class ParseIsoPipe {
    transform(dateString, options) {
        return parseISO(dateString, options);
    }
}
ParseIsoPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ParseIsoPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
ParseIsoPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ParseIsoPipe, name: "dfnsParseIso" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ParseIsoPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsParseIso' }]
        }] });
export class ParseIsoPipeModule {
}
ParseIsoPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ParseIsoPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
ParseIsoPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ParseIsoPipeModule, declarations: [ParseIsoPipe], exports: [ParseIsoPipe] });
ParseIsoPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ParseIsoPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: ParseIsoPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [ParseIsoPipe],
                    exports: [ParseIsoPipe]
                }]
        }] });
