// ngx-date-fns/esm2020/lib/start-of-month.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import startOfMonth from 'date-fns/startOfMonth';
import * as i0 from "@angular/core";
export class StartOfMonthPipe {
    transform(date) {
        return startOfMonth(date);
    }
}
StartOfMonthPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfMonthPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
StartOfMonthPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfMonthPipe, name: "dfnsStartOfMonth" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfMonthPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsStartOfMonth' }]
        }] });
export class StartOfMonthPipeModule {
}
StartOfMonthPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfMonthPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
StartOfMonthPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfMonthPipeModule, declarations: [StartOfMonthPipe], exports: [StartOfMonthPipe] });
StartOfMonthPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfMonthPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfMonthPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [StartOfMonthPipe],
                    exports: [StartOfMonthPipe]
                }]
        }] });
