// ngx-date-fns/esm2020/lib/start-of-week-year.pure.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import startOfWeekYear from 'date-fns/startOfWeekYear';
import { calculateLocale } from './date-fns-configuration.service';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class StartOfWeekYearPurePipe {
    constructor(config) {
        this.config = config;
    }
    transform(date, options) {
        return startOfWeekYear(date, calculateLocale(options, this.config));
    }
}
StartOfWeekYearPurePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekYearPurePipe, deps: [{ token: i1.DateFnsConfigurationService }], target: i0.ɵɵFactoryTarget.Pipe });
StartOfWeekYearPurePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekYearPurePipe, name: "dfnsStartOfWeekYearPure" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekYearPurePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsStartOfWeekYearPure' }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }]; } });
export class StartOfWeekYearPurePipeModule {
}
StartOfWeekYearPurePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekYearPurePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
StartOfWeekYearPurePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekYearPurePipeModule, declarations: [StartOfWeekYearPurePipe], exports: [StartOfWeekYearPurePipe] });
StartOfWeekYearPurePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekYearPurePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekYearPurePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [StartOfWeekYearPurePipe],
                    exports: [StartOfWeekYearPurePipe]
                }]
        }] });
