// ngx-date-fns/esm2020/lib/start-of-quarter.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import startOfQuarter from 'date-fns/startOfQuarter';
import * as i0 from "@angular/core";
export class StartOfQuarterPipe {
    transform(date) {
        return startOfQuarter(date);
    }
}
StartOfQuarterPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfQuarterPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
StartOfQuarterPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfQuarterPipe, name: "dfnsStartOfQuarter" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfQuarterPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsStartOfQuarter' }]
        }] });
export class StartOfQuarterPipeModule {
}
StartOfQuarterPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfQuarterPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
StartOfQuarterPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfQuarterPipeModule, declarations: [StartOfQuarterPipe], exports: [StartOfQuarterPipe] });
StartOfQuarterPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfQuarterPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfQuarterPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [StartOfQuarterPipe],
                    exports: [StartOfQuarterPipe]
                }]
        }] });
