// ngx-date-fns/esm2020/lib/start-of-decade.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import startOfDecade from 'date-fns/startOfDecade';
import * as i0 from "@angular/core";
export class StartOfDecadePipe {
    transform(date) {
        return startOfDecade(date);
    }
}
StartOfDecadePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfDecadePipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
StartOfDecadePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfDecadePipe, name: "dfnsStartOfDecade" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfDecadePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsStartOfDecade' }]
        }] });
export class StartOfDecadePipeModule {
}
StartOfDecadePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfDecadePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
StartOfDecadePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfDecadePipeModule, declarations: [StartOfDecadePipe], exports: [StartOfDecadePipe] });
StartOfDecadePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfDecadePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfDecadePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [StartOfDecadePipe],
                    exports: [StartOfDecadePipe]
                }]
        }] });
