// ngx-date-fns/esm2020/lib/start-of-minute.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import startOfMinute from 'date-fns/startOfMinute';
import * as i0 from "@angular/core";
export class StartOfMinutePipe {
    transform(date) {
        return startOfMinute(date);
    }
}
StartOfMinutePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfMinutePipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
StartOfMinutePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfMinutePipe, name: "dfnsStartOfMinute" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfMinutePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsStartOfMinute' }]
        }] });
export class StartOfMinutePipeModule {
}
StartOfMinutePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfMinutePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
StartOfMinutePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfMinutePipeModule, declarations: [StartOfMinutePipe], exports: [StartOfMinutePipe] });
StartOfMinutePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfMinutePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfMinutePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [StartOfMinutePipe],
                    exports: [StartOfMinutePipe]
                }]
        }] });
