// ngx-date-fns/esm2020/lib/start-of-tomorrow.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import startOfTomorrow from 'date-fns/startOfTomorrow';
import * as i0 from "@angular/core";
export class StartOfTomorrowPipe {
    transform() {
        return startOfTomorrow();
    }
}
StartOfTomorrowPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfTomorrowPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
StartOfTomorrowPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfTomorrowPipe, name: "dfnsStartOfTomorrow" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfTomorrowPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsStartOfTomorrow' }]
        }] });
export class StartOfTomorrowPipeModule {
}
StartOfTomorrowPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfTomorrowPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
StartOfTomorrowPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfTomorrowPipeModule, declarations: [StartOfTomorrowPipe], exports: [StartOfTomorrowPipe] });
StartOfTomorrowPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfTomorrowPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfTomorrowPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [StartOfTomorrowPipe],
                    exports: [StartOfTomorrowPipe]
                }]
        }] });
