// ngx-date-fns/esm2020/lib/start-of-yesterday.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import startOfYesterday from 'date-fns/startOfYesterday';
import * as i0 from "@angular/core";
export class StartOfYesterdayPipe {
    transform() {
        return startOfYesterday();
    }
}
StartOfYesterdayPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfYesterdayPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
StartOfYesterdayPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfYesterdayPipe, name: "dfnsStartOfYesterday" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfYesterdayPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsStartOfYesterday' }]
        }] });
export class StartOfYesterdayPipeModule {
}
StartOfYesterdayPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfYesterdayPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
StartOfYesterdayPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfYesterdayPipeModule, declarations: [StartOfYesterdayPipe], exports: [StartOfYesterdayPipe] });
StartOfYesterdayPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfYesterdayPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfYesterdayPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [StartOfYesterdayPipe],
                    exports: [StartOfYesterdayPipe]
                }]
        }] });
