// ngx-date-fns/esm2020/lib/start-of-hour.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import startOfHour from 'date-fns/startOfHour';
import * as i0 from "@angular/core";
export class StartOfHourPipe {
    transform(date) {
        return startOfHour(date);
    }
}
StartOfHourPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfHourPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
StartOfHourPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfHourPipe, name: "dfnsStartOfHour" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfHourPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsStartOfHour' }]
        }] });
export class StartOfHourPipeModule {
}
StartOfHourPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfHourPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
StartOfHourPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfHourPipeModule, declarations: [StartOfHourPipe], exports: [StartOfHourPipe] });
StartOfHourPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfHourPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfHourPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [StartOfHourPipe],
                    exports: [StartOfHourPipe]
                }]
        }] });
