// ngx-date-fns/esm2020/lib/start-of-day.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import startOfDay from 'date-fns/startOfDay';
import * as i0 from "@angular/core";
export class StartOfDayPipe {
    transform(date) {
        return startOfDay(date);
    }
}
StartOfDayPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfDayPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
StartOfDayPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfDayPipe, name: "dfnsStartOfDay" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfDayPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsStartOfDay' }]
        }] });
export class StartOfDayPipeModule {
}
StartOfDayPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfDayPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
StartOfDayPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfDayPipeModule, declarations: [StartOfDayPipe], exports: [StartOfDayPipe] });
StartOfDayPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfDayPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfDayPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [StartOfDayPipe],
                    exports: [StartOfDayPipe]
                }]
        }] });
