// ngx-date-fns/esm2020/lib/start-of-week.pipe.mjs
import { Pipe, NgModule } from '@angular/core';
import startOfWeek from 'date-fns/startOfWeek';
import { calculateLocale } from './date-fns-configuration.service';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class StartOfWeekPipe {
    constructor(config, cd) {
        this.config = config;
        this.cd = cd;
        this.localeChanged$ = this.config.localeChanged.subscribe(_ => this.cd.markForCheck());
    }
    ngOnDestroy() {
        this.localeChanged$.unsubscribe();
    }
    transform(date, options) {
        return startOfWeek(date, calculateLocale(options, this.config));
    }
}
StartOfWeekPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekPipe, deps: [{ token: i1.DateFnsConfigurationService }, { token: i0.ChangeDetectorRef }], target: i0.ɵɵFactoryTarget.Pipe });
StartOfWeekPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekPipe, name: "dfnsStartOfWeek", pure: false });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsStartOfWeek', pure: false }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }, { type: i0.ChangeDetectorRef }]; } });
export class StartOfWeekPipeModule {
}
StartOfWeekPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
StartOfWeekPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekPipeModule, declarations: [StartOfWeekPipe], exports: [StartOfWeekPipe] });
StartOfWeekPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [StartOfWeekPipe],
                    exports: [StartOfWeekPipe]
                }]
        }] });
