// ngx-date-fns/esm2020/lib/start-of-year.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import startOfYear from 'date-fns/startOfYear';
import * as i0 from "@angular/core";
export class StartOfYearPipe {
    transform(date) {
        return startOfYear(date);
    }
}
StartOfYearPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfYearPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
StartOfYearPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfYearPipe, name: "dfnsStartOfYear" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfYearPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsStartOfYear' }]
        }] });
export class StartOfYearPipeModule {
}
StartOfYearPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfYearPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
StartOfYearPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfYearPipeModule, declarations: [StartOfYearPipe], exports: [StartOfYearPipe] });
StartOfYearPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfYearPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfYearPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [StartOfYearPipe],
                    exports: [StartOfYearPipe]
                }]
        }] });
