// ngx-date-fns/esm2020/lib/start-of-week-year.pipe.mjs
import { Pipe, NgModule } from '@angular/core';
import startOfWeekYear from 'date-fns/startOfWeekYear';
import { calculateLocale } from './date-fns-configuration.service';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class StartOfWeekYearPipe {
    constructor(config, cd) {
        this.config = config;
        this.cd = cd;
        this.localeChanged$ = this.config.localeChanged.subscribe(_ => this.cd.markForCheck());
    }
    ngOnDestroy() {
        this.localeChanged$.unsubscribe();
    }
    transform(date, options) {
        return startOfWeekYear(date, calculateLocale(options, this.config));
    }
}
StartOfWeekYearPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekYearPipe, deps: [{ token: i1.DateFnsConfigurationService }, { token: i0.ChangeDetectorRef }], target: i0.ɵɵFactoryTarget.Pipe });
StartOfWeekYearPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekYearPipe, name: "dfnsStartOfWeekYear", pure: false });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekYearPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsStartOfWeekYear', pure: false }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }, { type: i0.ChangeDetectorRef }]; } });
export class StartOfWeekYearPipeModule {
}
StartOfWeekYearPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekYearPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
StartOfWeekYearPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekYearPipeModule, declarations: [StartOfWeekYearPipe], exports: [StartOfWeekYearPipe] });
StartOfWeekYearPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekYearPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekYearPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [StartOfWeekYearPipe],
                    exports: [StartOfWeekYearPipe]
                }]
        }] });
