// ngx-date-fns/esm2020/lib/start-of-second.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import startOfSecond from 'date-fns/startOfSecond';
import * as i0 from "@angular/core";
export class StartOfSecondPipe {
    transform(date) {
        return startOfSecond(date);
    }
}
StartOfSecondPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfSecondPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
StartOfSecondPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfSecondPipe, name: "dfnsStartOfSecond" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfSecondPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsStartOfSecond' }]
        }] });
export class StartOfSecondPipeModule {
}
StartOfSecondPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfSecondPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
StartOfSecondPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfSecondPipeModule, declarations: [StartOfSecondPipe], exports: [StartOfSecondPipe] });
StartOfSecondPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfSecondPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfSecondPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [StartOfSecondPipe],
                    exports: [StartOfSecondPipe]
                }]
        }] });
