// ngx-date-fns/esm2020/lib/start-of-iso-week-year.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import startOfISOWeekYear from 'date-fns/startOfISOWeekYear';
import * as i0 from "@angular/core";
export class StartOfISOWeekYearPipe {
    transform(date) {
        return startOfISOWeekYear(date);
    }
}
StartOfISOWeekYearPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfISOWeekYearPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
StartOfISOWeekYearPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfISOWeekYearPipe, name: "dfnsStartOfISOWeekYear" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfISOWeekYearPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsStartOfISOWeekYear' }]
        }] });
export class StartOfISOWeekYearPipeModule {
}
StartOfISOWeekYearPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfISOWeekYearPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
StartOfISOWeekYearPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfISOWeekYearPipeModule, declarations: [StartOfISOWeekYearPipe], exports: [StartOfISOWeekYearPipe] });
StartOfISOWeekYearPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfISOWeekYearPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfISOWeekYearPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [StartOfISOWeekYearPipe],
                    exports: [StartOfISOWeekYearPipe]
                }]
        }] });
