// ngx-date-fns/esm2020/lib/start-of-today.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import startOfToday from 'date-fns/startOfToday';
import * as i0 from "@angular/core";
export class StartOfTodayPipe {
    transform() {
        return startOfToday();
    }
}
StartOfTodayPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfTodayPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
StartOfTodayPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfTodayPipe, name: "dfnsStartOfToday" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfTodayPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsStartOfToday' }]
        }] });
export class StartOfTodayPipeModule {
}
StartOfTodayPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfTodayPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
StartOfTodayPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfTodayPipeModule, declarations: [StartOfTodayPipe], exports: [StartOfTodayPipe] });
StartOfTodayPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfTodayPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfTodayPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [StartOfTodayPipe],
                    exports: [StartOfTodayPipe]
                }]
        }] });
