// ngx-date-fns/esm2020/lib/start-of-iso-week.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import startOfISOWeek from 'date-fns/startOfISOWeek';
import * as i0 from "@angular/core";
export class StartOfISOWeekPipe {
    transform(date) {
        return startOfISOWeek(date);
    }
}
StartOfISOWeekPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfISOWeekPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
StartOfISOWeekPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfISOWeekPipe, name: "dfnsStartOfISOWeek" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfISOWeekPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsStartOfISOWeek' }]
        }] });
export class StartOfISOWeekPipeModule {
}
StartOfISOWeekPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfISOWeekPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
StartOfISOWeekPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfISOWeekPipeModule, declarations: [StartOfISOWeekPipe], exports: [StartOfISOWeekPipe] });
StartOfISOWeekPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfISOWeekPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfISOWeekPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [StartOfISOWeekPipe],
                    exports: [StartOfISOWeekPipe]
                }]
        }] });
