// ngx-date-fns/esm2020/lib/start-of-week.pure.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import startOfWeek from 'date-fns/startOfWeek';
import { calculateLocale } from './date-fns-configuration.service';
import * as i0 from "@angular/core";
import * as i1 from "./date-fns-configuration.service";
export class StartOfWeekPurePipe {
    constructor(config) {
        this.config = config;
    }
    transform(date, options) {
        return startOfWeek(date, calculateLocale(options, this.config));
    }
}
StartOfWeekPurePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekPurePipe, deps: [{ token: i1.DateFnsConfigurationService }], target: i0.ɵɵFactoryTarget.Pipe });
StartOfWeekPurePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekPurePipe, name: "dfnsStartOfWeekPure" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekPurePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsStartOfWeekPure' }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }]; } });
export class StartOfWeekPurePipeModule {
}
StartOfWeekPurePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekPurePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
StartOfWeekPurePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekPurePipeModule, declarations: [StartOfWeekPurePipe], exports: [StartOfWeekPurePipe] });
StartOfWeekPurePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekPurePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: StartOfWeekPurePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [StartOfWeekPurePipe],
                    exports: [StartOfWeekPurePipe]
                }]
        }] });
