// ngx-date-fns/esm2020/lib/sub-iso-week-years.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import subISOWeekYears from 'date-fns/subISOWeekYears';
import * as i0 from "@angular/core";
export class SubISOWeekYearsPipe {
    transform(date, amount) {
        return subISOWeekYears(date, amount);
    }
}
SubISOWeekYearsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubISOWeekYearsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
SubISOWeekYearsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubISOWeekYearsPipe, name: "dfnsSubISOWeekYears" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubISOWeekYearsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsSubISOWeekYears' }]
        }] });
export class SubISOWeekYearsPipeModule {
}
SubISOWeekYearsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubISOWeekYearsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
SubISOWeekYearsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubISOWeekYearsPipeModule, declarations: [SubISOWeekYearsPipe], exports: [SubISOWeekYearsPipe] });
SubISOWeekYearsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubISOWeekYearsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubISOWeekYearsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [SubISOWeekYearsPipe],
                    exports: [SubISOWeekYearsPipe]
                }]
        }] });
