// ngx-date-fns/esm2020/lib/sub-hours.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import subHours from 'date-fns/subHours';
import * as i0 from "@angular/core";
export class SubHoursPipe {
    transform(date, amount) {
        return subHours(date, amount);
    }
}
SubHoursPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubHoursPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
SubHoursPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubHoursPipe, name: "dfnsSubHours" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubHoursPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsSubHours' }]
        }] });
export class SubHoursPipeModule {
}
SubHoursPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubHoursPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
SubHoursPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubHoursPipeModule, declarations: [SubHoursPipe], exports: [SubHoursPipe] });
SubHoursPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubHoursPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubHoursPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [SubHoursPipe],
                    exports: [SubHoursPipe]
                }]
        }] });
