// ngx-date-fns/esm2020/lib/sub-weeks.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import subWeeks from 'date-fns/subWeeks';
import * as i0 from "@angular/core";
export class SubWeeksPipe {
    transform(date, amount) {
        return subWeeks(date, amount);
    }
}
SubWeeksPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubWeeksPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
SubWeeksPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubWeeksPipe, name: "dfnsSubWeeks" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubWeeksPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsSubWeeks' }]
        }] });
export class SubWeeksPipeModule {
}
SubWeeksPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubWeeksPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
SubWeeksPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubWeeksPipeModule, declarations: [SubWeeksPipe], exports: [SubWeeksPipe] });
SubWeeksPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubWeeksPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubWeeksPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [SubWeeksPipe],
                    exports: [SubWeeksPipe]
                }]
        }] });
