// ngx-date-fns/esm2020/lib/sub-seconds.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import subSeconds from 'date-fns/subSeconds';
import * as i0 from "@angular/core";
export class SubSecondsPipe {
    transform(date, amount) {
        return subSeconds(date, amount);
    }
}
SubSecondsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubSecondsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
SubSecondsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubSecondsPipe, name: "dfnsSubSeconds" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubSecondsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsSubSeconds' }]
        }] });
export class SubSecondsPipeModule {
}
SubSecondsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubSecondsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
SubSecondsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubSecondsPipeModule, declarations: [SubSecondsPipe], exports: [SubSecondsPipe] });
SubSecondsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubSecondsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubSecondsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [SubSecondsPipe],
                    exports: [SubSecondsPipe]
                }]
        }] });
