// ngx-date-fns/esm2020/lib/sub-months.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import subMonths from 'date-fns/subMonths';
import * as i0 from "@angular/core";
export class SubMonthsPipe {
    transform(date, amount) {
        return subMonths(date, amount);
    }
}
SubMonthsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMonthsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
SubMonthsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMonthsPipe, name: "dfnsSubMonths" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMonthsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsSubMonths' }]
        }] });
export class SubMonthsPipeModule {
}
SubMonthsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMonthsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
SubMonthsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMonthsPipeModule, declarations: [SubMonthsPipe], exports: [SubMonthsPipe] });
SubMonthsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMonthsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMonthsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [SubMonthsPipe],
                    exports: [SubMonthsPipe]
                }]
        }] });
