// ngx-date-fns/esm2020/lib/sub-milliseconds.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import subMilliseconds from 'date-fns/subMilliseconds';
import * as i0 from "@angular/core";
export class SubMillisecondsPipe {
    transform(date, amount) {
        return subMilliseconds(date, amount);
    }
}
SubMillisecondsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMillisecondsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
SubMillisecondsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMillisecondsPipe, name: "dfnsSubMilliseconds" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMillisecondsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsSubMilliseconds' }]
        }] });
export class SubMillisecondsPipeModule {
}
SubMillisecondsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMillisecondsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
SubMillisecondsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMillisecondsPipeModule, declarations: [SubMillisecondsPipe], exports: [SubMillisecondsPipe] });
SubMillisecondsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMillisecondsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMillisecondsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [SubMillisecondsPipe],
                    exports: [SubMillisecondsPipe]
                }]
        }] });
