// ngx-date-fns/esm2020/lib/sub-minutes.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import subMinutes from 'date-fns/subMinutes';
import * as i0 from "@angular/core";
export class SubMinutesPipe {
    transform(date, amount) {
        return subMinutes(date, amount);
    }
}
SubMinutesPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMinutesPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
SubMinutesPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMinutesPipe, name: "dfnsSubMinutes" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMinutesPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsSubMinutes' }]
        }] });
export class SubMinutesPipeModule {
}
SubMinutesPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMinutesPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
SubMinutesPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMinutesPipeModule, declarations: [SubMinutesPipe], exports: [SubMinutesPipe] });
SubMinutesPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMinutesPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubMinutesPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [SubMinutesPipe],
                    exports: [SubMinutesPipe]
                }]
        }] });
