// ngx-date-fns/esm2020/lib/sub-days.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import subDays from 'date-fns/subDays';
import * as i0 from "@angular/core";
export class SubDaysPipe {
    transform(date, amount) {
        return subDays(date, amount);
    }
}
SubDaysPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubDaysPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
SubDaysPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubDaysPipe, name: "dfnsSubDays" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubDaysPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsSubDays' }]
        }] });
export class SubDaysPipeModule {
}
SubDaysPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubDaysPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
SubDaysPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubDaysPipeModule, declarations: [SubDaysPipe], exports: [SubDaysPipe] });
SubDaysPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubDaysPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubDaysPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [SubDaysPipe],
                    exports: [SubDaysPipe]
                }]
        }] });
