// ngx-date-fns/esm2020/lib/sub-quarters.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import subQuarters from 'date-fns/subQuarters';
import * as i0 from "@angular/core";
export class SubQuartersPipe {
    transform(date, amount) {
        return subQuarters(date, amount);
    }
}
SubQuartersPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubQuartersPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
SubQuartersPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubQuartersPipe, name: "dfnsSubQuarters" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubQuartersPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsSubQuarters' }]
        }] });
export class SubQuartersPipeModule {
}
SubQuartersPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubQuartersPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
SubQuartersPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubQuartersPipeModule, declarations: [SubQuartersPipe], exports: [SubQuartersPipe] });
SubQuartersPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubQuartersPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubQuartersPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [SubQuartersPipe],
                    exports: [SubQuartersPipe]
                }]
        }] });
