// ngx-date-fns/esm2020/lib/sub-years.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import subYears from 'date-fns/subYears';
import * as i0 from "@angular/core";
export class SubYearsPipe {
    transform(date, amount) {
        return subYears(date, amount);
    }
}
SubYearsPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubYearsPipe, deps: [], target: i0.ɵɵFactoryTarget.Pipe });
SubYearsPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubYearsPipe, name: "dfnsSubYears" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubYearsPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsSubYears' }]
        }] });
export class SubYearsPipeModule {
}
SubYearsPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubYearsPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
SubYearsPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubYearsPipeModule, declarations: [SubYearsPipe], exports: [SubYearsPipe] });
SubYearsPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubYearsPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: SubYearsPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [SubYearsPipe],
                    exports: [SubYearsPipe]
                }]
        }] });
