// ngx-date-fns/esm2020/lib/utils/format-relative-to-now.pipe.mjs
import { Pipe, NgModule } from '@angular/core';
import { calculateLocale } from '../date-fns-configuration.service';
import formatRelative from 'date-fns/formatRelative';
import { isValidDate } from '../utils';
import * as i0 from "@angular/core";
import * as i1 from "../date-fns-configuration.service";
export class FormatRelativeToNowPipe {
    constructor(config, cd) {
        this.config = config;
        this.cd = cd;
        this.localeChanged$ = this.config.localeChanged.subscribe(_ => this.cd.markForCheck());
    }
    ngOnDestroy() {
        this.localeChanged$.unsubscribe();
    }
    transform(date, options) {
        if (isValidDate(date)) {
            const dateToCompare = Date.now();
            return formatRelative(date, dateToCompare, calculateLocale(options, this.config));
        }
        return '';
    }
}
FormatRelativeToNowPipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativeToNowPipe, deps: [{ token: i1.DateFnsConfigurationService }, { token: i0.ChangeDetectorRef }], target: i0.ɵɵFactoryTarget.Pipe });
FormatRelativeToNowPipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativeToNowPipe, name: "dfnsFormatRelativeToNow", pure: false });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativeToNowPipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsFormatRelativeToNow', pure: false }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }, { type: i0.ChangeDetectorRef }]; } });
export class FormatRelativeToNowPipeModule {
}
FormatRelativeToNowPipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativeToNowPipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
FormatRelativeToNowPipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativeToNowPipeModule, declarations: [FormatRelativeToNowPipe], exports: [FormatRelativeToNowPipe] });
FormatRelativeToNowPipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativeToNowPipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativeToNowPipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [FormatRelativeToNowPipe],
                    exports: [FormatRelativeToNowPipe]
                }]
        }] });
