// ngx-date-fns/esm2020/lib/utils/format-relative-to-now.pure.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import formatRelative from 'date-fns/formatRelative';
import { calculateLocale } from '../date-fns-configuration.service';
import { isValidDate } from '../utils';
import * as i0 from "@angular/core";
import * as i1 from "../date-fns-configuration.service";
export class FormatRelativeToNowPurePipe {
    constructor(config) {
        this.config = config;
    }
    transform(date, options) {
        if (isValidDate(date)) {
            const dateToCompare = Date.now();
            return formatRelative(date, dateToCompare, calculateLocale(options, this.config));
        }
        return '';
    }
}
FormatRelativeToNowPurePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativeToNowPurePipe, deps: [{ token: i1.DateFnsConfigurationService }], target: i0.ɵɵFactoryTarget.Pipe });
FormatRelativeToNowPurePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativeToNowPurePipe, name: "dfnsFormatRelativeToNowPure" });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativeToNowPurePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsFormatRelativeToNowPure' }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }]; } });
export class FormatRelativeToNowPurePipeModule {
}
FormatRelativeToNowPurePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativeToNowPurePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
FormatRelativeToNowPurePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativeToNowPurePipeModule, declarations: [FormatRelativeToNowPurePipe], exports: [FormatRelativeToNowPurePipe] });
FormatRelativeToNowPurePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativeToNowPurePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: FormatRelativeToNowPurePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [FormatRelativeToNowPurePipe],
                    exports: [FormatRelativeToNowPurePipe]
                }]
        }] });
