// ngx-date-fns/esm2020/lib/utils/weekday-name.pipe.mjs
import { NgModule, Pipe } from '@angular/core';
import eachDayOfInterval from 'date-fns/eachDayOfInterval';
import endOfWeek from 'date-fns/endOfWeek';
import format from 'date-fns/format';
import startOfWeek from 'date-fns/startOfWeek';
import { calculateLocale } from '../date-fns-configuration.service';
import * as i0 from "@angular/core";
import * as i1 from "../date-fns-configuration.service";
const formats = {
    x1: 'eeeee',
    x2: 'eeeeee',
    x3: 'eee',
    full: 'eeee'
};
export class WeekdayNamePipe {
    constructor(config, cd) {
        this.config = config;
        this.cd = cd;
        this.localeChanged$ = this.config.localeChanged.subscribe(_ => this.cd.markForCheck());
    }
    ngOnDestroy() {
        this.localeChanged$.unsubscribe();
    }
    transform(day, view = 'full', options) {
        const locale = calculateLocale(options, this.config);
        const now = new Date();
        const week = eachDayOfInterval({
            start: startOfWeek(now, locale),
            end: endOfWeek(now, locale)
        });
        return format(week[day], formats[view], locale);
    }
}
WeekdayNamePipe.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: WeekdayNamePipe, deps: [{ token: i1.DateFnsConfigurationService }, { token: i0.ChangeDetectorRef }], target: i0.ɵɵFactoryTarget.Pipe });
WeekdayNamePipe.ɵpipe = i0.ɵɵngDeclarePipe({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: WeekdayNamePipe, name: "dfnsWeekdayName", pure: false });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: WeekdayNamePipe, decorators: [{
            type: Pipe,
            args: [{ name: 'dfnsWeekdayName', pure: false }]
        }], ctorParameters: function () { return [{ type: i1.DateFnsConfigurationService }, { type: i0.ChangeDetectorRef }]; } });
export class WeekdayNamePipeModule {
}
WeekdayNamePipeModule.ɵfac = i0.ɵɵngDeclareFactory({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: WeekdayNamePipeModule, deps: [], target: i0.ɵɵFactoryTarget.NgModule });
WeekdayNamePipeModule.ɵmod = i0.ɵɵngDeclareNgModule({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: WeekdayNamePipeModule, declarations: [WeekdayNamePipe], exports: [WeekdayNamePipe] });
WeekdayNamePipeModule.ɵinj = i0.ɵɵngDeclareInjector({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: WeekdayNamePipeModule });
i0.ɵɵngDeclareClassMetadata({ minVersion: "12.0.0", version: "13.2.3", ngImport: i0, type: WeekdayNamePipeModule, decorators: [{
            type: NgModule,
            args: [{
                    declarations: [WeekdayNamePipe],
                    exports: [WeekdayNamePipe]
                }]
        }] });
